> # Learning Goals
> 1. Describe the three stages of Translation
> 2. Summarize the similarities and differences between translation in bacteria and in eukaryotes
> 3. Explain how some antibiotics inhibit the growth of bacteria by interfering with translation
>
> - Like transcription, the process of translation occurs in three stages
> 1. Initiation
> 2. Elongation
> 3. Termination
>
> - The image below provides the overview
> - Initiation, an mRNA, the first tRNA, and the ribosomal subunits assemble into a complex
> - In elongation, the ribosome moves in the 5` to 3 prime direction from the start codon in the mRNA towards the stop codon to make the polypeptide
> - Termination, the ribosome reaches the stop codon and the complex disassembles, releasing the completed polypeptide
>
> Now we observe how each step occurs
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/c9561a56-0537-4ce0-8192-934be6a38e05)


# Translation is Initiated with the Assembly of mRNA, tRNA, and the Ribosomal Sub units
- During **initiation** a complex is formed between the mRNA molecule, the first tRNA, and the ribosomal subunits
- All species has the complex requiring the help of proteins called initiation factors that facilitate the interactions between these components
- To assemble these requires an input of energy from Guanosine Triphosphate (GTP), being hydrolyzed by certain initiation factors to provide the energy needed
  - **When not involved in translation, the small and large ribosomal subunits exist separately**
- Here are the steps for assembly:
   1. To begin assembly of the complex, an mRNA binds to the small ribosomal subunit (image below), where the binding of mRNA to this subunit is facilitated by a **short ribosomal-binding site**—a stretch of nucleotides whose bases are complementary to those in a portion of the 16S rRNA in the small ribosomal subunits—near the 5` end of the mRNA. In the end, the mRNA becomes bound to the ribosome because the ribosomal-binding site and rRNA H-Bond to each other via base pairing, where the start codon is usualyl just a few nucleotides downstream (3 prime end) from the ribosomal-binding site.
    2. A specific tRNA, functioning as the initiator tRNA, recognizes the start codon (AUG) and binds to it; however, in eukaryotes, the tRNA carries methionine while bacteria carry methionine modified by attaching a formyl group
    3. Completing initiation happens when the large ribosomal subunit associates with the small subunit. At the end, the initiator tRNA is located at the P site of the ribosome.

> The Image of Translation initiation
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/87d5f495-1f87-4f6a-a499-76228183c859)

- Eukaryotes have the initiation phase differ from the translation phase in two ways compared to bacteria
  - Rather than an RNA sequence functioning as a ribosomal-binding site, eukaryotic mRNAs have a 7-methylguanosine cap (5` cap) at their 5 prime end
    - This 5` cap is recognized by cap-binding proteins that promote binding mRNA to the small ribosomal subunit
  - In bacterial, the start codon is close to the ribosomal-binding site, but the location of a start codon in eukaryotes are much more variable.
- In 1978, Marilyn Kozak proposed that small ribosomal subunits identifies a start codon by beginning at the 5` end of a sequence and the scanning along mRNA in the 3 prime direction to search for an AUG sequence
  - Analyzing the sequences of a lot of eukaryotic mRNA let Kozak and co. find that the sequence around an AUG codon lets is be recognized as a start codon.
  > Image showing how the sequence for an optimal start codon recognition occurs:
  >
  > ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/46d30e93-58e1-4a40-8234-bef729e8977b)
  - Aside from the AUG codon, a guanine just past the start codon and sequence of 6 bases directly upstream from the start codon are important for start codon selection
    - If the first AUG codon is within a site that deviates merkedly from this optimal sequence, the small subunit can **just skip this codon and use another AUG codon further down**
    - Once the small subunit finds a start codon, an initiator tRNA binds to the start codon, and then the large ribosomal subunit associates with the small subunit to complete the assembly process.


# Polypeptide Synthesis Occurs During the Elongation Stage
- Elongation involves the covalent bonding of amino acids to each other, one at a time to make a polypeptide
- This process needs several components, but occurs at a remarkable rate
  - Under normal conditions, the translation machinery can elongate a polypeptide at a rate of 15-18 amino acids per second in bacteria and 6 amino acids per second in Eukaryotes

# *tRNA Entry*
- Elongating a polypeptide requires a tRNA to bring in a new amino acid to the ribosome where the growing polypeptide is
- Step 1 shows how translation already proceeded to a point where a short polypeptide is attached to the tRNA in the P site of the ribosome
  - The resulting structure is called a peptidyl tRNA
  - The first step of elongation, a charged tRNA carrying a single amino acid binds to the A site, binding because the anticodon in the tRNA is complementary to the codon in the mRNA
  - Hydrolysis of GTP by proteins functioning as elongation factors provides the energy for the binding of the tRNA to the A site
  - At this step, a peptidyl tRNA is in the P site and a charged tRNA (an aminoacyl tRNA) is in the A site.

> Image of the elongation stage of transcription
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/725ee2f5-ff3a-4886-97e0-39f6a1cba83f)

# *Peptidyl Transfer Reaction*
- In the second step, a peptide bond is formed between the amion acid at the A site and the growing polypeptide, lenthening the polypeptide by one amino acid
- As this happens, the polypeptide is removed from the tRNA in the P site and transferred to the amino acid at the A site, aka **peptidyl transfer reaction**
- This reaction is catalyzed by an rRNA molecule within a region of the 50S subunit known as the peptidyltransferase center
- 2000, Thomas Steitz and Peter Moore proposed that the rRNA is responsible for catalyzing the peptide bond formation between adjacent amino acids
- It is now accepted that the ribosome is a ribozyme

# *Translocation of the Ribosome and Release of tRNA*
- After the peptidyl transfer reaction is complete, the third step involves the movement (**Translocation**) of the ribosome toward the 3` end of the mRNA by exactly 1 codon.
  - This shifts the tRNAs in the P and A site to the E and P sites (RESPECTIVELY)
  - Then the uncharged tRNA exits the E site
  - Now that the next codon in the mRNA (GCU in the image above) now has an exposed and unoccupied A site, where now a charged tRNA can enter the A site, and the same series of steps will add the next amino acid to the polypeptide
- In the image above, you can see a single ribosome in the process of translating an mRNA
  - In living cells, it is common for multiple ribosomes to glide along the same mRNA and synthesize polypeptides
  - **The complex of a single mRNA and multiple ribosomes is called a polysome**

# Termination Occurs When a Stop Codon is Reached in the mRNA
- Elongation continues until a stop codon moves into the A site of a ribosome
- The three stop codons, UAA, UAG, and UGA are recognized by a protein called a **release factor**, where the 3d structure mimics the structure of tRNAs, letting it fit into the A site
  - Caveman fight: UAA! *attacks grug with club** UAG! *grug cries in pain* UGA! *sug shouts victory cheer*
- The image below easily illustrates the termination of translation
    1. A release factor binds to the stop codon at the A site. The completed polypeptide is attached to a tRNA in the P site.
    2. The bond between the polypeptide and the tRNA is hydrolyzed, causing the polypeptide and tRNA to be released from the ribosome
    3. The mRNA, ribosomal subunits, and release factor dissociates
> Image of termination in question
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/5f763cda-6fa2-4ea8-a121-8085aa34baf7)

- Termination stage is similar in bacteria and eukaryotes except bacteria have 2 different transcription factors that recognize stop codons (RF1 and RF2) whilst eukaryotes only have one (eRF)
- Table below compares key difference between bacterial and eukaryotic translation

> Table in question:
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/2a0fb039-009f-4a5c-ac46-29f088867ad6)

# Antibiotics That Inhibit Bacterial Translation Are Used to Treat Bacterial Infections
- Many diseases affecting humans and domesticated animals are caused by **pathogenic bacteria**
- **antibiotics** is a substanced produced by a microorganism to inhibit the growth of other microorganisms, like the pathogenic bacteria
  - Most antibiotics are small, organic molecules, with masses less than 2e3 Da
- Some antibiotics inhibit/interfere bacterial translation
- As the components of translation differ somewhat between bacteria and eukaryotes, some antibiotics inhibit bacterial translation without affecting eukaryotic translation, so they can be used on humans, pets, and livestock

> Table describing a few antibiotics and how they inhibit bacterial translation:
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/59ae9893-0c88-4c07-89a5-734e14463456)
