> # Learning Goals:
> 1. Explain how regulatory transcription factors and small effector molecules are involved int the regulation of transcription
> 2. Describe the organization of the *lac* operon, and explain how it is under negative and positive control
> 3. Analyze Jacob, Monod, and Pardee's results
> 4. Explain how the *trp* operon is under negative control
>
> We've seen bacterium exposed to a nutrient in an envrionment expressing genes needed to encode proteins to metabolize sugar. we also seen bacteria have genes encoding enzymes to synthesize molecules like amino acids. These genes require the control of expression occuring at the level of transcription. THis section examines the molecule mechnisms bringing about transcriptional regulation in bacteria.

# Transcriptional Regulation Involves Regulatory Transcription Factors and Small Effector Molecules
- Most cases, regulation of transcription involves the actions of **Regulatory Transcription Factors**, proteins that bind to regulatory sequences in the DNA, usually found in the vicinity of the promoter affecting rate of transcription of one or more nearby genes
- Transcription factors can increase or decrease rate of transcription
- Repressors are regualtory transcription factors that bind to the DNA and decrease the rate of transcription
  - This is negative control
- Activators bind to the DNA and increase the rate of transcription
  - A form of tested **positive control**

> Actions of regulatory transcription factors and small effector molecules
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/3ea5785b-bc10-4d82-ac95-7aaf47c1e260)

- Molecules called **small effector molecules** play a role in transcriptional regulatory
  - They exert its effects via binding to a regulatory transcription factor and causing conformational change in the protein
  - Many cases, the effect of the conformational change determines whether or not the protein can bind to the DNA
- Above, we see a repressor in the (b) cateogry
  - When the small effect molecule is not in the cytoplasm, the repressor binds to the DNA and ihibits transcription
  - When a small effector molecule is there, it binds to the repressor and causes conformational change inhibiting the ability of the protein to bind to the DNA
  - Transcription can occur because the repressor is not able to bind to the DNA
- Repressors and activators repsonding to small effector molecules have two functional regions called domains
  - One domain is a site where protein binds to the DNA
  - The other is for the small effector molecule

# The *lac* Operon Contains Genes Encoding Proteins Involved in Lactose Metabolism
- In bacteria, 2 or more genes can be under transcriptional control of a single promoter
- This arrangement is an **operon**
  - The genes in the operon are a single unit, resulting in production of polycistronic mRNA, an mRNA encoding more than one protein
  - This gives the advantage of coordinating regulation of genes that encode proteins whose functions are used in common pathways
- The genome of E. coli carries an operon called the ***lac* operon** (lactose operson) containing the genes to metabolize lactose
- Figure below shows the organization of this operon found in E. coli chromosome, and the polycistronic mRNA transcribed from it
- The *lac operon* contains a promoter *lacP*, involved in the transcription of three protein-coding genes: *lacZ, lacY, and lacA*
  - *lacZ* gene encodes Beta-galactosidase, an enzyme breaking down lactose. It also converts a small percentage of lactose into allolactose, a structurally similar sugar, or lactose analogue. Allolactose is important in the regualtion of the *lac* operon
  - *lacY* encodes lactose permease, membrane protein needed to transport lactose into the cytoplasm of bacterium
  - *lacA* gene encodes galatoside transacetylase, covalently modifies lactose and lactose analogues by attaching an acetyl group (-COCH<sub/>3</sub>). Attachment of acetyl groups to nonmetabolizable lactose analogues prevents toxic buildup in the cytoplasm

- Near the *lac* promoter are 2 regulatory sequences designated the oerpator and the CAP site (check out (a) below)
  - The operator (*lacO*) is a regulatory sequence in the DNA
  - The sequence of bases at the operator provides a binding site for a repressor protein
  - The **CAP site** is a regulatory sequence recognized by an activator protein.
- Adjacent to the *lac* operon is the *lacI* gene, encoding the lac repressor
  - This repressor is important in regulation of the *lac* operon
  - The *lacI* gene, constitutively expressed at a low level, has the promoter called the *i* promoter
  - The *lacI* gene is not considered a part of the *lac* operon

> The *lac* operon
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/d5732ab8-b691-41ed-b3fc-f1b35e049315)

# The *lac* Operon is Under Negative Control by a Repressor Protein
- The late 1950s Jacob and Monod at France focused on gene regulation due to the phenomenon of enzyme adaptation
- Enzyme adaptation occurs when a particular enzyme appears in a living cell only when the substrate is exposed to it
- Jacob and Monod studied lactose metabolism in E. coli to investigate this phenomenon, exposing bacteria to lactose
  - Lactose-utilizing enzymes increased by 1e3-1e4 times the normal amount, and after lactose was removed, synthesis stopped.
- The first mechanism of regulation they found involved lac repressors, binding to the sequenes of bases at the *lac* oeprator site
  - Binding results in prevention of RNA polymerase from transcribing *lacZ,Y, and A* genes
  - RNA polymerase can bind to the promoter when the lac repressor is bound to the operator site, but cannot move past the operator to transcribe the Z, Y, and A

> Negative control of an inducible set of genes: function of the lac repressor in regulating the *lac* operon
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/a18f2924-dd7e-4499-b4f3-88371e1cd0f5)

- Whether or not the lac respssor binds to the operator site depends on allolactose, the side product of Beta-galactosidase enzyme
  - allolactose controls the lac repressor by being a small effector molecule
  - The lac repressor protein contains 4 identical subunits, each recognizing a single allolactose molecule
  - When the 4 bind to the lac repressor, a conformation change occurs preventing repressor from binding to the oeprator
  - Under these condition, RNA polymerase is free to transcribe the operon
- Regulation of the *lac* operon enables E. coli to conserve energy as lactose-utilizing protiens are made only when lactose is present in the environment
- Allolactose is an **inducer**, a small effector molecule increasing transcription rate, and the *lac* operon is an **inducible operon**
- When the bacterium is not exposed to lactose, no allolactose is available to bind to the lac repressor
  - Thus, lac repressors binds to the operator site and inhibits transcription
  - In reality, the repressor doesn't inhibit it completely, as small amounts of Beta-galactosidase, lactose permease, and galactoside transacetylase are made
  - But not enough to readily use lactose
- When bacterium is exposed to lactose, small amounts can be transported into the cytoplasm via lactose permease and Beta-galactosidase converts some to allolactose
  - The cytoplasmic level of allolactose gradually rises until allolactose binds to the lac repressor, inducing the *lac* operon and promoting high rates of transcription of the *lacZ, Y, and A* genes
  - Translation of the encoded polypeptides produces the proteins needed for lactose uptake and metabolism

# Feature Investigation | Jacob, Monod, and Pardee Studied a Constitutive Mutant to Deterine the Function of the Lac Repressor
- As it turns out, lac repressors bind to the *lac* operator site to exert its effects
- How did they figure this out?
- They used *lac* operson regulation coming from studies involving E. coli strains showing abnormalities in  the process
- Jacob, Monod, and co and Pardee found few mutants expressing *lac* operson constitutively, so *lacZ, Y, and A* were expressed even in the absence of the lactose in environment
  - They discovered some mutations caused this and occurred in the *lacI* region of the DNA
  - Such strands are *lacI<sup/>-</sup>* to indicate non-functionality
  - Normal is *lacI<sup/>+</sup>*
- Initially thought *lacI<sup/>+</sup>* encodes an enzyme that degrades an internal inducer of the *lac* operson*
  - *lacI<sup/>-</sup>* inhibits the enzyme, letting continuous synthesis
  - *lacI<sup/>-</sup>* made is unnecessary for cells to be exposed to lactose for induction
- Then they found that *lacI* encodes a repressor protein, so *lacI* genes just eliminates the synthesis of the functional lac repressor and prevents the lac repressor protein from inhibiting transcription

> Hypothesis for the function of the *lacI* gene
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/ea2307fb-8eae-4b46-a80d-52476b650a48)

- To understand this *lacI<sup/>-</sup>*, all three applied a genetic approach using bacterial genetic exchange via F factors
  - Some F factors also carry genes originally found within the bacterial chromosome
  - These types are called F<sup/>'</sup> factors (F prime factors)
  - A cell having F<sup/>'</sup> factors is called a **merozygote**
  - Study of these let the three elucidate the function of the *lacI* gene
- These researchers studied the *lac* operon in a bacterial strain carrying a *lacI<sup/>-</sup>*h mutation causing constitutive expression of the *lac* operon
  - Furthermore, the F<sup/>'</sup> factor was transferred to the mutant strain, making a merozygote carrying a normal *lac* operon and normal *lac* operon and normal *lacI<sup/>+</sup>* gene on this F' factor
  - The merozygote contained both *lacI<sup/>+</sup>* and *lacI<sup/>-</sup>* genes
- The constitutive mutant and corresponding merozygote were grown separately in liquid media and divided into two tubes each
  - halaf the tubs had cells with lactose, other tube lactose was omitted
  - They monitored the expression by breaking open the cell and testing them for the amount of Beta-galatctosidase released by measuring the ability of any Beta-galactosidase present to convert a colorless compound into a yellow product
- The data table below summarizes the effects of constitutive mutation and its analysis in a merozygote
  - The mutant *lacI<sup/>-</sup>* strain expressed *lac* operon constitutively, presence and absence of lactose
  - But when the normal *lac* operon and *lacI<sup/>+</sup>* gene on a F' factor was introduced into a cell harboring the mutant *lacI<sup/>-</sup>* gene on the chromosome, the normal *lacI<sup/>+</sup>* gene could regulate both operons
  - No lactose, both operons were shut off
- Explain by saying a single *lacI<sup/>+</sup>* gene on the F' factor can make enough repressor proteins to bind both operator sites
  - The protein is also diffusable and can spread through the cytoplasm, binding to the *lac* operon that are on F' factors and on bacterial chromosomes
  - Together, the data indicates normal *lacI* gene encodes diffusible protiens repressing the *lac* operon
- This made 3 new genetic terms
  - **cis-acting element** is a DNA segment adjacent to the gene it regulates (*lac* operator site)
  - **trans-effect** is a form of gene regulation occuring even if 2 DNA segments aren't physically adjacent (lac repressor on lac operon)
  - **cis-effect** is mediated by a cis-acting element binding regulatory proteins, whereas the trans-effect is mediated by genes encoding diffusible regulatory proteins

> Table in question
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/f0de81b5-d9f4-4404-b935-bd00636b53e9)


# The *lac* Operon is Also Under Positive Control by an Activator Protein
- The *lac* operon is also positively regulated by an activator called the **caatabolite activator protein (CAP)**
  - Controlled by small effector molecule, cyclic AMP (from ATP via adenylyl cyclase I mean remember fucking epinephrine?)
  - Gene regulation involving CAP and cAMP is examples of a Positive control
  - When cAMP binds to CAP, this complex binds to the CAp site near the *lac* promoter
  - This causes a bend in the DNA enhancing the ability of RNA polymerase to bind to the promoter
  - Thus, rate of transcription is increased

> Positive control of the *lac* operon by the cataboolite activator protein (CAP)
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/1d3d9982-eaa7-496f-867b-672fa3066d0c)

- The function of CAP is to let E. coli choose between different sugars as an energy source, **Catabolite repression**
  - Presence of preferred energy source inhibits the use of other energy sources
  - Transcription of the *lac* operon is inhibited by the presence of glucose, a catabolite (broken down in the cell)
  - This lets E. coli preferentially use glucose instead of other sugars like lactose
- HOW?
  - Glucose inhibits cAMP production and prevents binding of CAP to the DNA
  - Glucose blocks activatino of *lac* operon
  - catabolite repression was coined before the action of cAMP-CAP complex was actually understood molecularly
  - We just saw glucose inhibited lactose metabolism
- Figure below shows 4 possible environmental conditions an *E. coli* bacterium might experience with regard to these two sugars
  - High lactose and glucose shows CAP not binding to its site, inhibiting transcription
    - A low level of transcription still occurs
    - bacterium primarily uses glucose instead of lactose
    - Why? Energy conservation
  - If the lactose level is high and glucose low, transcription rate of the *lac* operon is high
    - CAP is bound to the CAP site and the lac repressor is not bound to the operator site
    - Bacterium metabolizes lactose
  - Lactose low, lac repressor prevents transcription of the *lac* operon, whether glucose is high or low

> Effects of lactose and glucose on the expression of the *lac* operon
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/0d989d99-a2c7-42c7-8c44-3f4e92a4f05a)

# The *trp* Operon is Under Negative Control by a Repressor Protein
- This section, we saw the regulation of the *lac* operon. Lets now see how an operon encoding enzymes involved in biosynthesis rather than breakdown
- We see ***trp* Operon* of E. Coli, coding enzymes required to make tryptophan, a building block of proteins (*trpE, trpD, trpC, trpB, and trpA* more specifically)
- The *trp* operon is regulated by a repressor protein encoded by the *trpR* gene
  - Binding of the repressor to the *trp* operator site inhibits transcription
  - The ability for the trp repressor to binde to the *trp* operator is controlled by trptophan, the product of the metabolic pathway controlled by the enzymes encoded by the operon
- Tryptophan conditions checklist:
  - If Tryptophan levels low, trp repressor cannot bind to the operator site, so RNA polymerase readily transcribes the operon and the cell expresses the genes encoding enzymes so tryptophan is synthesized
  - When it is high, the *trp* operon is turned off, so Tryptophan is a small effector molecule, or **corepressor** by binding to the trp repressor protein
    - Causes conformatino change in repressor so it binds to the *trp* operator site, inhibiting ability of RNA polymerase to transcribe the operon so bacterium does not waste energy making trptophan when it is abundant

> Negative control of the repressible set of genes: fucntion of the trp repressor and corepressor (tryptophan) in regulating *trp* operon
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/e79fb572-a0e3-4f14-ace7-aafcc1c37c2a)


- The actions of small effector molecules are quite different for *lac* and *trp* operons
  - lac repressor binds to the operator in absence, trp binds in presence
  - *lac* is categorized as an inducible operon due to allolactose inducing transcription
  - *trp* operon is a repressible oeprson as its small effector molecule, tryptophan, represses transcription
