> # Learning Goals:
> 1. Describe flanking of eukaryotic gene by nucleosome-free regions, explaining how nucleosomes are altered during gene transcription
> 2. Explain how DNA methylation affects transcription
> 3. Describe how formation of facultative heterochromatin can silence genes in a tissue-specific manner
>
> DNA found in chromosomes is associated with proteins and non-coding RNAs to form chromatin
>
> But how does it really affect gene transcription? well from Chatper 11 nucleosomes are DNA wrapped in an octomer in histone proteins so the location and arrangement of nucleosomes may be in a closed conformation and transcription could be difficult to impossible
>
> Transcription requires changes in chroamtin structure to allow transcription factors to gain access to and bind to the DNA in the promoter region, and such chromatin is in the **open conformation**, accessible to GTFs and RNA polymerase II so transcription could take place
>
> This seciton covers how chromatin is converted from a closed to an open conformation, exploring how DNA methylation, attachment of methyl groups to cytosine, affects chromatin conformation and gene expression

# Transcription is Controlled by Changes in Chromatin Structure
- Recent years, geneticists tried identifying steps promoting interconversion betwen the closed and open conformations of chromatin
- One way to change chromatin is via ATP-dependent chromatin-remodeling complexes, complexes of protein altering chromatin structure that use energy from ATP hydrolysis to drive a change in the locations or compositions of nucleosomes making DNA more or less amenable to transcription
- Chromatin remodeling is important for both actvation and the repression of transcription
- How do ATP-dependent chromatin-remodeling complexes change chromatin structure
  - The complexes may bind to chromatin and change the location of nucleosomes, shifting relative positions of a few nucleosomes or changing the relative spacing of nucleosomes over a long stretch of DNA
  - Remodeling complexes may evict histone octamers from DNA, creating gaps where nucleosomes are not found
  - Chromatin-remodeling complexes may change the composition of nucleosomes by removing standard histone proteins from an octamer and replacing them with histone variants, a protein with slightly different amino acid sequences than that of the standard histone protein described in Chapter 11 (some promote transcription, some inhibit it)

 > ATP-dependent chromatin remodeling
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/04160163-3d64-4096-ab73-c9d78b0f7c20)

# Histone Modifications Affect Gene Transcription
- Recently, researchers found amino terminal tails of histone proteins are subject to many covalent modifications
- E.g. Enzyme histone acetyltransferase attaching acetyl groups (-COCH<sub/>3</sub>) to an amino terminal tail of histone proteins
  - Acetylation results in histone proteins not binding as tightly to DNA, aiding in transcription
- Over 50 enzymes selectively modify amino terminal tails found in mammals
- Below, we see how amino terminal tails of H2A, H2B, H3, and H4 can be modified by attachment of acetyl, methyl, and phosphate groups

> Examples of covalent modifications of the amino terminla tails of histone proteins
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/63c176be-b6c0-42fa-bfd5-a9ceedc9124c)

- Effects of covalent histone modification
  - Can influence interactinos of DNA and histone proteins and between adjacent nucleosomes (acetylation loosens binding to DNA)
  - Histone modifications provide binding sites recognized by other proteins
- **Histone Code Hypothesis**, Histone modification is recognized by proteins much like a language or code
  - One pattern attracts proteins inhibiting transcription
  - Another pattern attracts ATP-dependent chromatin-remodeling complexes, promoting gene transcription
  - Histone code plays a key role in determining the accessibility of information within the genomes of eukaryotic species

# Eukaryotic Genes Are Flanked by Nucleosome-Free Regions
- Studies over the last 10 years or so revealed many eukaryotic genes show a common pattern of nucleosome organization
- For active genes or those than can be activated, the core promoter is near the **nucleosome-free region (NFR)**, a site in the chromatin that is missing nucleosomes
- NFR Characteristics
  - The NFR is typically 150bp in length
  - NFR is needed for transcription, but isn't sufficient for gene activation. Anytime in a life of a cell, many genes containing an NFR are not being actively transcribed
  - The NFR is flanked by 2 nucleosomes termined -1 and +1 nucleosomes, which are histone variants promoting transcription
  - The end of many eukaryotic genes is followed by another NFR, the arrangement at the end of genes maybe being important for transcriptional termination

> Nucleosome arrangements in the vicinity of a eukaryotic protein-coding gene
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/c4d1443a-719d-448b-8fb2-380b7118b295)

# Transcriptional Activation Involves Changes in Nucleosome Locations and Changes in Histones
- A key role in certain activators is to recruit ATP-dependent chroamtin-remodeling complexes and histone-modifying enzymes to the promoter region of eukaryotic genes
- Order of recruit can be different among specific activators, the recruiting appears critical for trasncriptional initiation and elongation
- Shown below, this scenario has the activator bind to the enhancer in the NFR

> A Simplified model for the transcriptional activation of a eukaryotic protein-coding gene
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/c9b283eb-425e-4110-8d2f-cbe5cc701612)


- The activator then recruits chromatin-remodeling compelxes and histone-modifying enzymes to this region
  - Chromatin-remodeling complexes shifts nucleosomes or temporarily evict nucleosomes from the promoter region
  - Nucleosomes containing certain histone variants are thought to be more easily removed from teh DNA than those containing standard histones
  - Histone-modifying enzymes, like acetyltransferase, covalently modifies histone proteins and may affect nucleosome contact with DNA
  - The actions of chromatin-remodeling complexes and histone-modifying enzymes facilitate the binding of general transcription factors and RNA polymerase II to the core promoter, allowing formation of a preinitiation complex
- Further changes in chromatin structure are necessary for elongation to occur
  - RNA polymerase II cannot transcribe DNA that is tightly wrapped in nucleosomes
  - To transcribe, histones are evicted, displaced, or destabilized so RNA polymerase II can pass
  - Evicted histones are then reassembled by chaperone proteins and put back into the DNA behind the moving RNA polymerase II
  - These histones may be deacetylated so they bind more tightly to DNA

# DNA Methylation Inhibits Gene Transcription
- DNA structure can be modified by attachment of methyl groups via DNA methyltransferase
- DNA methylation is common in some eukaryotic species but not all
  - E.g. Yeast and Fruit flies have no methylation in DNA
  - Vertebrates and plants have relatively abundant methylation
  - 5% methylated in mammals, and Eukaryotic DNA methylation occurs on the cytosine base

> Sequence of methylated DNA
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/08c2581a-a8d2-461e-9cf0-b62919984e00)

- Methylation usually inhibits transcription, especially when it is near the promoter
- Vertebrates and flowering plants have many genes containing CpG islands near promoters
  - CpG refers to Cytosine, Guanine in DNA whose nucleotides are connected by a phosphodiester linkage, called an island as it is a cluster of them
  - Unmethylated CpG islands are usually correlated with active genes, methylated the reverse
  - DNA methylation thus silences particular genes
- How does Methylation inhibit transcription?
  - Methylation of CpG islands can prevent an activator from binding to an enhancer element, inhibiting the initiation of transcription
  - It can also alter chromatin structure via **methyl-CpG-binding proteins** binding to methylated sequences. Once bound, the protein "recruits to the site other proteins that inhibit transcription" (exact quote. Idk what the fuck the book is doing)

# The Formation of Facultative Heterochromatin Is a way to Silence Genes in a Tissue-Specific Manner
- Chromosomal regions in nondividing cells can occur as euchromatin or heterochromatin
  - Compactness of heterochromatin silences gene expression by preventing transcription
  - certain regions of heterochromatin also differ among different types
  - This form of heterochroamtin is facultative hterochromatin
    - E.g. DNA in muscle cell can occur within facultative heterochromatin and contain several genes whose expression is silences
    - In a neuron, the same DNA may be packaged in euchromatin and the genes within the segment will not be silences
    - This way, interconversinos between euchromatin and heterochromatin play a key role in silencing genes in a tissue-specific manner
- During embryonic development, certain regions of chromosomes are targeted to become facultative heterochromatin
  - This involves specific types of histone modifications and DNA methylation of CpG islands
  - These events cause chromatin to become more compact and serves to silence genes in such a region
- After facultative heterochromatin has formed during embryonic development, the same region remains heterochromatic following cell division
  - E.g. if a region in aembryonic muscle cell is heterochromatic, the same region will be heterochromatic in all daughter cells derived from the embryonic cell
  - This way, formation of facultative heterochromatin can silence gene expression in a tissue-specific manner

> Pattern of heterochromatin before and after cell division
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/50b748c4-1954-4fac-ac0d-25258a4d5ee8)

