> # Learning Goals:
> 1. Describe flanking of eukaryotic gene by nucleosome-free regions, explaining how nucleosomes are altered during gene transcription
> 2. Explain how DNA methylation affects transcription
> 3. Describe how formation of facultative heterochromatin can silence genes in a tissue-specific manner
>
> DNA found in chromosomes is associated with proteins and non-coding RNAs to form chromatin
>
> But how does it really affect gene transcription? well from Chatper 11 nucleosomes are DNA wrapped in an octomer in histone proteins so the location and arrangement of nucleosomes may be in a closed conformation and transcription could be difficult to impossible
>
> Transcription requires changes in chroamtin structure to allow transcription factors to gain access to and bind to the DNA in the promoter region, and such chromatin is in the **open conformation**, accessible to GTFs and RNA polymerase II so transcription could take place
>
> This seciton covers how chromatin is converted from a closed to an open conformation, exploring how DNA methylation, attachment of methyl groups to cytosine, affects chromatin conformation and gene expression

# Transcription is Controlled by Changes in Chromatin Structure
- Recent years, geneticists tried identifying steps promoting interconversion betwen the closed and open conformations of chromatin
- One way to change chromatin is via ATP-dependent chromatin-remodeling complexes, complexes of protein altering chromatin structure that use energy from ATP hydrolysis to drive a change in the locations or compositions of nucleosomes making DNA more or less amenable to transcription
- Chromatin remodeling is important for both actvation and the repression of transcription
- How do ATP-dependent chromatin-remodeling complexes change chromatin structure
  - The complexes may bind to chromatin and change the location of nucleosomes, shifting relative positions of a few nucleosomes or changing the relative spacing of nucleosomes over a long stretch of DNA
  - Remodeling complexes may evict histone octamers from DNA, creating gaps where nucleosomes are not found
  - Chromatin-remodeling complexes may change the composition of nucleosomes by removing standard histone proteins from an octamer and replacing them with histone variants, a protein with slightly different amino acid sequences than that of the standard histone protein described in Chapter 11 (some promote transcription, some inhibit it)

 > ATP-dependent chromatin remodeling
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/04160163-3d64-4096-ab73-c9d78b0f7c20)

# Histone Modifications Affect Gene Transcription
- 
