> # Learning Goals:
> 1. Distinguish between genomics and functional genomics
> 2. Outline the steps of DNA sequencing using the dideoxy chain-termination method
> 3. Explain how a DNA microarray and RNA-Seq are used to identify the genes that are transcribed by a sample of cells
> 4. Describe how CRIPR-Cas technology can be used to edit genes.
> 5. Explain how gene therapy works
>
> As said throughout Unit 3, the genome is the complete genetic composition of a species. Genomics is the molecular analysis of the entire genome of a species. Segments of chromosomes are cloned and analyzed in progressively smaller pieces, locations of which are known on the intact chromosomes (this is the mapping phase of genomics). Mapping is proceeded with complete DNA sequence determination, the most detailed description of the organism's genome. **Functional genomics** just studies experssion of a genome, checking which genes are on or off in cancer cells. This section considers a few methods used in genomics and functional genomics

# The Dideoxy Chain-Termination Method Is Used to Determine the Base Sequence of DNA
- **DNA Sequencing** refers to the procedure aimed at determining the base sequence of DNA
- One types by Sanger and co is **dideoxy chain-termination method**, or dideoxy dsequencing
  - DNA polymerase connects adjacent deoxynucleoside triphosphates (dNTPs) by catalyzing covalent linkage between phospahte groups at the 5` position on the incoming nucleotide and the -OH group at the 3 prime position on the growing strand
  - Or a chemist can just synthesize a nucleotide called dideoxynucleoside triphosphates (ddNTPs) missing the -OH group at the 3` position
  - The ddNTP being added a growing DNA strand causes the strand to no longer grow due to the lack of the 3` -OH Group. This is **chain termination**
 
> Structure of a dideoxynucleotide triphosphate
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/90c57626-7cc2-4d3e-bc76-b0e8c550a49d)

- What DNA segment even is analyzed here though?
  - This is the target DNA, obtained in large amounts via gene cloning LAST chapter (not linking it fuck you)
  - Below, the target DNA is inserted next to a primer-annealing site, the place where a primer will bind
  - Initially double-stranded DNA, the DNA is soon denatured into a single strand 

> Dideoxy sequencing of DNA (Detailed Procedure Below)
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/76846c4b-1a45-4615-b57f-d91204369fea)

> Fun fact: ddG is yellow, but black on a printout because fuck yellow on white

- Steps of DNA sequencing (REFER TO IMAGE ABOVE)
  - Many copies of single-stranded template DNA are placed into a tube, mixed with primers binding to the primer-annealing site. DNA polymerase and all four types of dNTPs are also added with LOW AMOUNTS of ddNTPs of all types (ddGTP, ddATP, ddTTP, ddCTP) each being tagged with a different colored fluorescent molecules (A green, T red, G yellow, C blue)
  - The tube is incubated, the DNA polymerase synthesizing strands complementary to the target DNA sequence. But ddNTP terminates it early, preventing further elongation of the strand. We then can see its fluorescently glowing green, red, yellow, etc for A,T,G, etc.
  - After the samples have been incubating, newly made DNA strands are separated according to length, either using slab gel, or a gel-filled capillary tube. Shorter strands move to the bottom more quickly than longer ones, and electrophoresis is continued until each band emerges from the bottom of the gel, where a laser excites the fluorescent dye letting a fluorescence detector record number of fluorescence emissions at four wavelengths, corresponding to the four dyes

- Above, looking at the (b) section, one sees the fluorescence detector
  - Peaks show DNA sequence complementary to the target DNA
  - Heights aren't the same, as ddNTPs get incorporated more easily at some sites than others
  - This usually makes a sequence of 700-900 bases long, or even longer
- Researchers are also developing alternative methods to the dideoxy sequencing method in order to sequence DNA
  - These things now superseed this sequencing method, and now considered the next-generation sequencing method
  - We got optical detection now, sequence single DNA molecules, makes it unnecessary to make many copies of template DNA, nanopore sequencing to just read it directly

# A DNA Microarray and RNA-Seq Are Methods to Identify Which Genes Are Transcribed by a Cell
- **Transcriptome*8 is the set of all RNA molecules (mRNAs and ncRNAs) transcribed in one or in a population of cells
- They may focus on identification of each type of RNA molecules and relative concentrations
- This section focuses on the two methods used to analyze these

## *DNA Microarrays*
- **DNA Microarray** is a small clear slide dotted iwth many different sequences of single-stranded DNA, each corresponding to a short sequence within a known gene
  - Each spot contains copies of a known DNA sequence, e.g. one being Beta-globin gene, another to a differnt one, and one to a glucose transporter
  - A single slide contains tens of thousands of dots in the size of a postage stamp, made with a printer that stamps holes in
- Why DNA microarray?
  - We figure out the genes transcribed into mRNA from a particular sample of cells
- mRNA is first isolated from cells and used to make fluorescently labeled cDNAs
  - cDNAs then incubated with the microarray, and the single-stranded DNA in the microarray corresponds to the coding strand, the strand having the sequence similar to mRNA
  - cDNAs complementary to the DNAs in the microarray hybridize, remaining binding to the microarray
  - The array is washed to remove unbound cDNAs, then a microscope equipped with a computer scans all spots and generates an image of relative fluorescence

- High fluorescence means a large number of cDNA was in the sample, hybridized to the DNA in the location
- This only identifies which mRNAs or ncRNAs are made though, as they just degrade at different rates so amount doesn't matter

> DNA microarray diagram
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/94de3460-fdac-4f27-983b-87a21db42cf7)

## *RNA-Seq*
- RNA Seqeuncing (RNA-Seq) made by Michael Snyder in 2008
  - we can see general steps here, but some changes based on types of RNA molecules studied and methods of next-generation sequencing used
- After the desired population of RNas have been obtained, the RNA is fragmented
  - The next step is to produce cDNAs via reverse transcriptase, the population of cDNAs is then subjected to DNA sequencing, amking a diverse colelction of cDNA sequences
- Finally, you compare the collection of cDNA sequences iwth already known genome sequence of athe organism from which the RNA was isolated
  - cDNA sequences are aligned with genomic DNA sequence, accomplished with computer technology
  - When cDNA is aligned with a gene sequence within the genome, the result means the gene was expressed, as each cDNA sequence is derived from an RNA molecule
  - Furthermore, the number of times a particular cDNA sequence aligns iwth a gene is a measurement of the gene's expression level
  - E.g. shown below, more cDNAs are obtained from gene A vs gene B as gene A was expressed at a higher level
- RNA seq can also identify introns, and the alignment of cDNAs with genomic DNA lets researchers see pattern of RNA splicing


> Technique of RNA Sequencing
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/0be7de13-8bd0-4d24-85d9-c7b496bbfeaf)
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/4ed1c738-8dd3-419a-bd2e-058e388d66c8)


- The most common use of DNA microarrays and RNA-seq is to study gene expression patterns

> Obligatory table
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/38a25cc5-1ff8-42ad-a9c9-4f7e2dfeb2bf)

# Genes in Living Cells Can Be Altered Using CRISPR-Cas Technology
- Yeah it can do **gene editing**
- Modifications were made
  - We first create a single RNA made of a tracrRNA and crRNA covalently linked to one another
  - The single guide RNA (sgRNA) and its spacer regions are designed to be complementary to one of the DNA strands of a target gene the researcher wants to edit
  - The sgRNA binds to the protein Cas9 and guides it to the target gene, letting Cas9 double-strand break a gene
- The break leads to two different DNA repair events
  - If done via end joining, the gene can get a small deletion, inactivating it, especially if a frameshift mutation is present in the coding sequence
  - If a researcher creates and adds a double-stranded segment of donor DNA (with a particular mutation that the researcher wants), homologous to the region where the break occurs, a different DNA system swaps in the donor DNA by a double crossover, letting specific mutations be put in a gene


> The use of CRISPR-Cas Technology for Gene Editing
>
> ![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157453648/02502a20-ba3c-4d75-88d3-245029a68bfa)

# Gene Therapy Can Be Used to Treat Human Disease
- **Gene Therapy**






