> # Learning Goals
> 1. Describe the Structure of Nucleosomes and the 30-nm fiber, and explain how the 30-nm fiber forms loop domains
> 2. Outline the various levels of compaction that lead to a metaphase chromosome
>
> - Typical eukaryotic Chromosomes contain single, linear, double-stranded DNA molecules that could be 10^8 base pairs in length
> - Chromosomes means a discrete unit of genetic material
  > - Human somatic cells contain 46 chromosomes
> - Chromatin is the complex of DNA and Proteins that makeup Eukaryotic Chromosomes
> - Chromosomes are dynamic structures alternating in tight and loose states
> - Here, we will see how chromosomes are compacted into the cell's nucleus and how the additional compaction is necessary to produce highly condensed chromosomes during cell division

# DNA Wraps Around Histone Proteins to Form Nucleosomes
- Histones can wrap and compact DNA
- Repeating structural units of Eukaryotic chromatin is the nucleosome, 11nm in diameter at the widest point
  - Each nucleosome is 146-147 base pairs of DNA wrapped around an octamer of histone proteins
    - 1. H2A, 2. H2B, 3. H3, 4. H4
    - Histone proteins are basic proteins with a large number of positively charged amino acids (lysine arginine)
      - The negative charges in the phosphates of DNA are attracted to the positive charges on histone proteins
      - The amino tail of each histone protein protrudes from the histone octamer
      - These tails can be covalently modified playing a key role in gene regulation
    - Nucleosomes are linked by linker regions of DNA (20-100 base pairs in length based on species and cell type)
    - A histone called H1 is then bound to the linker region along with other types of proteins
    - This structure resembles beads on a string, and shortens the length of DNA molecules by 7 fold

> Image of a Nucleosome's Structure
>
> <img width="378" alt="Screenshot 2024-02-18 at 9 27 17 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/45383f93-fd79-4233-95c2-669d34b03c69">


# Nucleosomes Form a 30 nm FIber
- Nucleosome units are organized in more compact 30nm diameter structures, also known as the 30-nm fiber
- Histone H1 and other proteins are important in forming the 30-nm fiber that can shorten the nucleosome structure another 7 fold
- The structure of the 30-nm fiber is difficult to determine as the conformation of the DNA may be substantially altered when extracted from living cells.
- Rachel Horowitz-Scherer and Christopher Woodock in 1990 suggested that nucleosmoes zigzag back and forth with a straight linker region

> Image of zig zaggers
>
> <img width="380" alt="Screenshot 2024-02-18 at 9 31 39 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/8e70a9b1-2574-4234-85c4-aec540344d26">

# Chromosomes Are Further Compacted by the Formation of Loop Domains
- To fit inside a cell's nucleus, the 30-nm fiber needs to be in loops, called **loop domains**.
- The protein CCCTC binding factor (CTCF) binds to three regularly spaced repeats of the core sequence CCCTC
  - In the image below, Two different CTCF's can bind to the DNA and then bind to each other to form a DNA loop
- Another mechanism is facilitated by a category of proteins called SMC proteins (Structural Maintenance of Chromosomes)
  - SMC proteins form a dimer that can wrap itself around two segments of DNA to form a loop
    - Image of Dimer EXAMPLE:
    -![image](https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/a5d0df1f-5f6f-495b-8ce6-bd0b3a385732)

  - Additional loop formation is required during interphase so chromosomes can fit inside of a cell's nucleus
  - SMC proteins are also involved in chromosome condensation and sister chromatid cohesion occurring in mitosis
- Researchers have found SMC proteins are found in the vicinity of CTCFs
  - SMC proteins may wrap around sites bound together by CTCF dimers (image below shows)
  - Why? Likely because CTCF dimers need loop formation and then subsequent association of SMC proteins to prevent the loop from coming apart

>Image showing the formation of a loop Domain via CTCF *or* SMC protein
>
> <img width="382" alt="Screenshot 2024-02-18 at 9 36 07 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/4bc72ae3-17f8-43c3-8033-bff41e7a3b9e">

#### How are these dumbass chromosomes organized within the cell nucleus?
  - Nondividing cells have each chromosome occupy its discrete region in the cell nucleus that usually does not overlap with the territories of adjacent chromosomes, so different chromosomes are not intertwined even when in a noncompacted conditions
  - The compaction levels are also not completely uniform
    - The variability can be seen with light microscopes, first discovered by Emil Heitz in 1928
      - These are **Heterochromatin** to describe the highly compacted regions of chromosomes during interphase
      - The less condensed are known as **Euchromatin** (form of chromatin in which the 30-nm fiber forms loop domains)
      - Heterochromatin have the loop domains compacted even further, where Genes found in heterochromatin are not able to be transcribed into RNA
      - The interconversion of euchromatin and heterochromatin is one mechanism to regulate the expression of genes

# During Cell Division, Chromosomes Undergo Maximum Compaction
- Division cues chromosomes to be more compacted or condensed
- The compaction aids in proper alignment in metaphase
- The image below shows the levels of compaction contributing to the metaphase chromosomes
  - DNA in the nucleus is always compacted by forming nucleosomes and condensing into 30-nm fiber
  - Euchromatin has the 30-nm fibers in a loop domain that are loose, letting ample space be between the 30-nm fibers
    - The average width of such loops are 300nm

> Image showing the steps of Eukaryotic Chromosomal Compaction
>
> <img width="643" alt="Screenshot 2024-02-18 at 9 47 38 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/6a446590-c803-4967-921a-2e62e7147cbc">

### THIS SHIT IS A MYSTERY
- Despite innovations in light, electron microscopy, computer modeling, etc. The detailed organization and compaction of metaphase chromosomes is largely a mystery
- The condensation process generally involves a closer association of loop domains with each other and a closer association of adjacent nucleosomes
- During cell division, chromosome condensation is so much tighter that little space is between the 30-nm fibers
  - All euchromatin is highly condensed, shortening chromosomes
- A metaphase chromosomes with two copies of DNA (image above) has a width of ~1400 nm, but the length is much shorter than the same chromosome in the nucleus of a nondividing cell

## CONTROVERSY: Are Nonhistone proteins forming a scaffold to Organize the shapes of Metaphase chromosomes
- The chromosome scaffold was originally identified as a proteinaceous structure that could be derived from metaphase chromosomes after the removal of histone proteins
- Biochemical analysis shows that scaffolds have large amounts of SMC proteins, so is there a possible relation?
- They found two SMC proteins (condensin and cohesin) playing a critical role in chromosomal condensation and sister chromatid cohesion
- But many researchers in the field do not believe that interconnected protein scaffolds play an important role in organizing metaphase chromosomes so more research is needed.
