> # Learning Outcomes:
> 1. Described the Structure and Function of tRNA
> 2. Explain how aminoacyl-tRNA synthetases attached amino acids to tRNAs
> 3. Outline the structural features of bacterial and eukaryotic ribosomes
> 4. Analyze how ribosomal RNA (rRNA) is used to evaluate evolutionary relationships among different species
> - Translation requires more componenets as the sequence of codons in an mRNA molecule must be translated as a sequence of amino acids according to the genetic code
> - A single protein cannot accomplish such a task, instead, many different proteins and RNA molecules interact in intricate steps to synthesize a polypeptide
> - Using mRNA, tRNA, ribosomes, and translation factors, a cell can synthesize polypeptides
>
> Table describing the components of Translation Machinery
>
> <img width="689" alt="Screenshot 2024-02-27 at 7 43 50 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/47f5ceb7-6296-47b9-af2a-e20f37eea538"> 
>
> - Cells use an unholy and substantially absurd amount of energy to translate mRNA into polypeptides.
>   - E.g. E Coli uses 90% of their cellular energy for this process
> - Translation is therefore important as hell, so this section sees the components of the translation MACHINERY and then what happens in the stages of translation as they happen in living cells

# Transfer RNAs Share Common Structural Features
- To see how tRNAs function as carriers of the correct amino acids during translation, researchers have examined their structural characteristics.
- The tRNAs of bacteria, archaea, and eukaryotes share common features
- Proposed in 1965 by Robert Holley, the 2d structure of a tRNA molecule resembles a cloverleaf
  - The structure of a tRNA has 3 stem-loops with a fourth stem with a 3` single-stranded region (shown below)
  - The stem is the region where RNA is double-stranded due to complementary base pairing via hydrogen bonds, and the loops are areas without base pairing (THE PARALLEL AREAS ARE HYDROGEN BONDS WITH COMPLEMENTARY PAIRING AND THE 3 LOOPS HAVE NON!
  - The anticodon is located at the loop of the middle step-loop region (BOTTOM MOST LOOP)
  - The 3` single-stranded region is the amino acid attachment site (EXTRA STICKOUT AREA)
  - The 3 structure of tRNA molecules involve additional folding of the secondary structure

> Image of the structures of tRNA
>
> <img width="453" alt="Screenshot 2024-02-27 at 7 53 28 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/f7a22417-a72a-470a-bca0-a0b70f6834a1">

- Cells in every organism can make many different tRNA molecules encoded by different genes
- tRNA is named according to the amino acid is carried, so tRNA<sup>Ser</sup> has serine in it
  - Due to genetic code containing 6 different serine codons, the cell can produce more than one type of tRNA<sup>Ser</sup>

# Aminoacyl-tRNA Synthetases Charge tRNAs by Attaching an Appropiate Amino Acid
- To perform its role during translation, a tRNA must have the appropriate amino acid attached to its 3` end.
- Aminoacyl-tRNA synthetases catalyze attaching amino acids to tRNA molecules
- Cells makes 20 different aminoacyl-tRNA, each type recognizing one of the 20 different amino acids
- Each of the aminoacyl-tRNA synthetase recognize alanine and attaches this aminoa cid to all tRNAs with alanine anticodons
- Aminoacyl-tRNA synthetase catalyze chemical reactions involving an amino acid, a tRNA molecule, and ATP
    1. A specific amino acid and ATP bind to the Enzyme
    2. The amino acid is activated by the covalent attachment of adenosine monophosphate (AMP) and pyrophosphate is released
    3. The activated amino acid is covalently attached to the 3` end of a tRNA molecule and AMP is released
    4. The tRNA and its amino acid, a charged tRNA, or an aminoacyl tRNA, is released from the enzyme

> Image of the process of tRNA charging
>
> <img width="700" alt="Screenshot 2024-02-27 at 8 29 13 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/b8b14cb7-ce69-4088-adf3-30e44815f0d7">

- **The second genetic code** is the ability of each aminoacyl-tRNA synthetase to recognize their appropriate tRNA
- A precise recognition process is necessary to maintain the fidelity of genetic information
- If the wrong amino acid was attached to a tRNA, the amino acid sequence of the translated polypeptide would be WRONG
  - Thankfully, aminoacyl-tRNA synthetases are stupidly accurate, only getting it wrong 1/100,000 times.
  - The anticodon region of the tRNA is usually important for recognition by the correct aminoacyl-tRNA synthetase
  - The base sequences in other regions may facilitate binding to an aminoacyl-tRNA synthetase

# Ribosomes Are Composed of rRNAs and Proteins
- **Ribosomes** do translation, and are the molecular machine
- Bacterial cells have one type of ribosome, translating all mRNAs in the cytoplasm
- EUKARYOTIC CELLS HOWEVER are compartmentalized into cellular organelles, biochemically distinct regions ribosomes are found in different cellular compartments
  - The most abundant type of eukaryotic ribosome functions in the cytosol.
  - Mitochondria also have ribosomes, and plant and algal cells have ribosomes in the chloroplasts
    - Unless otherwise noted, the term eukaryotic ribosomes refers to ribosomes in the cytosol, not those found in other organelles
- The ribosome is a large complex composed of structures called the large and the small subunits
  - The term subunit is also misleading, as each ribosomal subunit is assembled from ANY proteins and one or more RNA molecules, called ribosomal RNAs, or rRNAs
    - In the case of E. coli, the small ribosomal subunit is called the 30S, and the large subunit is 50S (SHOWN BELOW)
    - 30S and 50S refer to the rate where subunits sediment when subjected to a centrifugal force
      - This is the sedimentation coefficient in Svedberg units named after a swede, Theodor Svedberg making the ULTRACENTRIFUGE
    - The 30S subunit made of 21 different ribosomal proteins and one 16S rRNA molecule
    - The 50S subunit contains 34 different proteins and 2 different rRNA molecules, called 5S and 23S
    - TOGETHER! The 30S and 50S subunits make the 70S ribosome (Svedberg units don't add up linearly as the sedimentation coefficient is a function of the size and shape of the object)
- Eukaryotic ribosomes consist of subunits that are slightly larger than their bacterial counterparts (shown below)
  - 40S and 60S subunits combine to make the 80S ribosome
    - The 40S is comprised of 33 proteins and an 18S rRNA
    - The 60S subunit has 49 proteins and 5S, 5.8S and 28S rRNAs
  - Synthesizing eukaryotic rRNAs happens in the nucleolus, a droplet organelle in the nucleus that is specialized for this purpose
  - The ribosomal proteins are made in the cytosol and are imported in the nucleus
  - The rRNAs and ribosomal proteins are then assembled within the nucleolus to make 40S and 60S subunits
  - The 40S and 60S subunits are exported into the cytosol, where they associate to form an 80S ribosome during translation

> Image of the composition of Bacterial and Eukaryotic Ribosomes
>
> <img width="694" alt="Screenshot 2024-02-27 at 8 43 25 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/5fc8d897-5e24-44ab-b556-dd1599792106">


# Components of Ribosomal Subunits Form Functional Sites for Translation
- Understanding the structure and function of a ribosome at the molecular level required researchers to determine the locations and functional roles of individual ribosomal proteins and rRNAs
- Recently, research groups succeeded in purifying ribosomes and crystalizing them in a test tube
- When researchers use X-ray diffraction to study the crystallized ribosomes, they obtain detailed information about ribosome structure.

> Image of a model of Bacterial ribosome
>
> <img width="710" alt="Screenshot 2024-02-27 at 8 46 22 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/60d36a19-4abd-4203-9df7-1faa25761989">

- During Bacterial translation, the mRNA lies on the surface of the 30S subunit, in a space between the 30S and 50S subunits, in a space between the 30S and 50S subunits
- As the polypeptide is synthesized, it exits through a hole within the 50S subunit
- Ribosomes have discrete sites where tRNAs bind and the polypeptide is synthesized
- 1964, James Watson proposed a 2 site model for tRNA binding to the ribosome, the peptidyl site (P site) and aminoacyl site (A site)
- 1981, Knud Nierhaus and Hans-Jorg Rheinberger turned it into a 3-site model (Image above, section b)
- Third site is the exit site (E site) and NEXT CHAPTER WE SEE THE ROLES OF THESE SITES IN SYNTHESIZING POLYPEPTIDES!

# Core Concept: Evolution

## _Comparisons of Small Subunit rRNAs Among Different Species Provide a Basis for Establishing  Evolutionary Relationship_
- Translation is a fundamental process vital for existing as a living species
- Components for translation occurred very early in the evolution of life on our planet in our last common ancestor
- For this reason, all organisms have translational components evolutionarily related to each other
  - E.g. the rRNA found in the small subunit of ribosomes is similar in all forms of life
  - Though it is larger in eukaryotic species (18S0 than bacterial species (16S)
  - The gene for the small subunit rRNA (SSU rRNA) is found in the genomes of all organisms
- Why is this useful?
  - The way geneticists explore evolutionary relationships is to compare sequences of evolutionarily related genes
  - At the molecular level, gene evolution involves changes in DNA sequences
  - After 2 different species have diverged from each other during evolution, the genes of each species change via mutations that alter the sequences of those genes
  - After many generations, the species contain genes that are similar but not identical due to mutations, so if two species diverged a long time ago, they would be different genetically, but if recently, their genes are still more similar
- Below compares portions of SSU rRNA genes from 3 mammalian and 3 bacterial species
  - Colors show differences, with bases shaded in yellow as identical in 5 or six species
    - This similarity between species is considered evolutionarily conserved
    - These sequences were likely found in a primordial gene giving rise to our modern species
  - Sequences in green are similar to all mammals, differing from bacterial species
    - For this case, we see identical sequences in this region
  - The sequences in red are identical or very similar in bacterial species, but differ from the mammalian SSU rRNA genes
- Sequences from the _E. Coli_ and _Serratia marcescens_ are more similar to each other than the sequence from _Bacillus subtilis_ is to either of them
  - Therefore, _E. Coli_ and _Serratia marcenscens_ are more closely related evolutionarily than _B. subtilis_

> Image comparing the SSU rRNA gene sequences of 6 species
>
> <img width="644" alt="Screenshot 2024-02-27 at 8 57 52 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/6c9e6809-ae78-4556-9cfe-949c6e3a2d14">

