 Thank you. This cold or whatever it is that I've got and have had for the last three weeks not only isn't going away, it's getting worse. This last weekend was as bad as it's been. That part's true. So on Sunday, I decided that I just didn't have the energy to make a full meal for my kids, so I decided to heat up some leftovers. And they just complained, complained, complained. So eventually I just threw them out, and then I sat down and had my leftovers. Good afternoon, everyone. Happy Monday to you all. Welcome to exam week in MCB 150. We'll see how well adrenaline, and adrenaline alone, gets me through this class period. I realistically should not be here today. That's how much I care. Remember that when you're filling out your evaluations at the end of the semester. I really do love being here. It is exam week. Let's talk about the events of exam week. Remember that today, every Monday of exam week at 5 p.m. is the deadline for letting us know that you want a conflict exam, and then you'll hear probably tomorrow where and when that conflict exam will be. Today's content will be on Thursday night's exam. We will have a regular lecture on Wednesday, but that material will not be on Thursday's exam. Wednesday's lecture material begins exam three material. I will have my regular student hours for your last-minute questions on Wednesday from 4 to 5.30 as always, and also as always, the TAs are around for help sessions on Mondays, Wednesdays, and Thursdays, so you still have plenty of opportunities to get real help from real people before Thursday night's exam. The exam two is going to be the same as exam one in terms of structure. It's 50 questions, multiple choice, Scantron. You've been through it. You know what to expect. There are going to be no curveballs there. Obviously, the content will be different, but the structure of the exam will be exactly the same. And then to compensate you for asking you to come in for an evening exam, we're going to give you Friday off, which is a nice little bonus because that may help you get out of here a little bit earlier for spring break. I do want to point out that, yes, every class period has a post-lecture assignment. And usually, usually, the policy is that post-lecture assignment will be due at 1 p.m. the next business day. Yes, there will be a post-lecture assignment for Wednesday's class, but I don't want you to have to worry about that while you're getting ready for Thursday's exam since that material isn't on Thursday's exam. So I'm not going to make it due until Monday when we come back from spring break. So you'll have plenty of time. You can get in there and do it any time you want, but you'll have until 1 p.m. on the Monday after spring break to do the post-lecture questions for lecture 21 and the pre-lecture questions for lecture 22. All right. Now, where did we leave off? Sort of in the middle of a story. We were talking about how do different organisms identify an initiator codon. We learned that in the bacterial world, there's a sequence in an mRNA called a Scheindl-Garno sequence or a ribosome binding site that is complementary and anti-parallel to a sequence in the 16S rRNA component of that 30S ribosomal subunit. And you find that sequence, and oh, look what's right there, an AUG. That's the one you use. What about eukaryotes? How do eukaryotes find the appropriate initiator codon? Eukaryotes don't use Scheindl-Garnold sequences, ribosome binding sites. They take advantage of our friend the 5' cap. We talked a little bit earlier about the reason why eukaryotes put a 5' cap on their mRNAs. This is one of those reasons. The small ribosomal subunit in a eukaryote, which this time is called a 40S subunit, not a 30S subunit, the 40S subunit just grabs onto a 5' cap, scans down, and looks for the first A next to a U next to a G that it encounters. And for purposes of MCB150, that's how eukaryotes identify an initiator codon. This also has a limitation built into it. If what you're looking for, in terms of finding an initiator codon, is a 5' cap, how many 5' caps can a molecule have? Well, another way to ask that question is how many 5' ends does a nucleic acid have? One, right? If you've only got one 5' end, you can only have one 5' cap, which means you can only recognize one thing and scan down until you find the first AUG. So, unlike the bacterial mRNAs, which have the potential of being polycistronic, eukaryotic mRNAs are always going to be monocistronic. And this is why all of the processing had to be done before these mature mRNAs were exported out into the cytoplasm. So we've got there, here's our 5' cap, it gets recognized, you scan down until you find an AUG, then you go through the coding region until you find a stop codon, and you can only do that for one polypeptide. Eukaryotic mRNAs, a specific eukaryotic mRNA, can only code for one polypeptide. If you want to do alternative splicing... Go ahead. But what it means is that you're going to prepare different monocystronic mRNAs with different combinations of exons in them. In the end, you can only get one polypeptide made from a single eukaryotic mRNA. So just to summarize what we just talked about, the way bacteria do it, I don't know why there's a typo there, this is Dalgarno, not Delgarno. The bacteria find this sequence in the mRNA, and oh look, what's about ten bases away? An AUG. This one is how eukaryotes do it, the 40S subunit recognizes the one and only 5' cap. This, by the way, is how you don't have to worry about accidentally trying to translate a tRNA or an rRNA. They won't have 5' caps on them. You are dependent on the 5' cap to then go looking for the nearest AUG. No 5' cap, not even going to attempt to translate that molecule. This is important. In determining a reading frame, finding the right AUG determines the reading frame. I've got a simplistic way and then a science-y way of demonstrating what I mean by reading frame. Here's a sentence that's made of simple three-letter words, and all of those words make sense to us. The fat cat ate the red rat. Because I'm reading three letters at a time, and they all make sense to me. But what if I chose to not start my three-letter sequence with the T, and instead start my three-letter sequence with the H? Then I'm going to go H-E-F, A-T-C, A-T-A. That doesn't make any sense whatsoever, because I chose the wrong reading frame. The same information is there, but you have to find the right reading frame in order for it to make sense. So again, that's the simplistic view. Now let's take a look at a more biological view. Here's a more biological view. What if I have three AUGs that could potentially be initiator codons? Would I make the same ultimate protein product from all three of them? Let's say, what would the protein product be if I used the first AUG? If this was my choice of initiator codon, then I know AUG specifies methionine, the next bases in a row are CAU, then GCG, then ACA, then UGG, and so forth. You will have a copy of a codon table, stapled to the back of your exam packet, you do not have to memorize what any codon specifies, just look it up in the table, and if you look up those codons, you see that this polypeptide would start with the sequence methionine, histidine, alladine, threonine, and so forth. That's the polypeptide made if I choose AUG number one in this figure as my initiator codon. Would I make the same polypeptide if I chose AUG number two? So I still start with methionine, of course, because all polypeptides in vivo have to start with methionine. But I'm already changing things. Now it's CGA, CAU, GGC, and so forth. And if I look these up, look these codons up in a codon table, they don't make anywhere near the same polypeptide. In fact, the only thing that polypeptide number one has in common with polypeptide number two is that they both start with methionine, which was a given. They're completely different polypeptides. One of them is likely to be the right thing for the cell, and the other one is going to be nonsense. It's going to not make anyâ€”why did you put these amino acids together? I can't use this thing. And then a more extreme example, what if I chose this one as my initiator codon? I've got AUG, GCU, CGG, UAA, and UAA is one of the three stop codons. So I would terminate after only three amino acids. Almost certainly not what I intended to do. So you make very different polypeptides based on your choice of initiator codon. That's why it's so important to pick the right one. And then we've initiated. Now elongation is going to follow initiation. Elongation is a cycle of three repeated steps that are going to keep going until we encounter a stop codon. Step number one, binding of the next charged tRNA molecule. Step number two, peptide bond formation. Step number three, the movement of the ribosome, three nucleotides relatively closer to the three prime end of the mRNA. I say relatively closer because whether you want to think of it as the ribosome staying put and the mRNAs being pulled through or the... mRNA is held in place and the ribosome is sliding down it. You can find examples of both. It doesn't matter. The relative result is the same. The ribosome ends up getting three bases closer to the three prime end of the mRNA and that process is called translocation. So let's watch these three repeated steps in the elongation phase of translation. The first one is the binding of the next charge tRNA. Here's our initiation complex that we saw last Friday. We're ready to bring in the next charge tRNA into the open A site once we have an available A site. We didn't have an A site before the large subunit joined the party. Now we're ready to actually do translation. What we're going to do is bring in the next charge tRNA and allow its anticodon to attempt to base pair with the codon that's in that A site. That's a nice picture. It shows continuity with the figures I was showing before, but it doesn't show an important part of the process. So I'm going to move it out of the way and show you a slightly different version of the same thing that's a little more holistic in its representation. This one, the previous figure, we were adding amino acid number two to amino acid number one. We hadn't really even gotten started yet. This one, we're adding one, two, three, four, five, six, seven, eight, nine, ten. We're adding amino acid. We're going to add amino acid number 12 to amino acid number 11. That's the only real difference here. Notice that the charged tRNA does not come in by itself. Remember that a lot of these processes need help in the form of factors, proteins that assist. Well, we're going to need help in the elongation phase, and surprisingly, or not surprisingly, that help is going to come in the form of elongation factors. In bacteria, this would be called EFTU. In eukaryotes, it would be called EF1. Don't worry about the difference. They do the same thing. Notice what they're doing here. What I want to do, once I identify that this is the right tRNA for the job, you have to, let me back up a second, any charged tRNA can take a shot at what's going on in the A site. And 19 times out of 20, it's going to be wrong, right? Most of the time, that's not going to be the right tRNA molecule. Only one out of 20 times is it going to be the right one. So we need a way to not screw something up. If the wrong tRNA gets in there. So if any charged tRNA gets in there, notice that the amino acid it's carrying has been bent over and away from the protein chain that we would like to attach it to. We're not going to let the ribosome create a covalent bond between amino acid number 12 and amino acid number 11 until we've verified that the codon and anticodon pair appropriately. If they do, if there is good fit between codon and anticodon, then the elongation factor hydrolyzes its GTP. That energy is used to peel the elongation factor off of the tRNA, and it's discarded. It just leaves the ribosome. And now the amino acid on that tRNA springs back into its normal position, and it's right next to amino acid number 11. Now, because we've verified that there is codon-anticodon pairing, we allow the elongation factor to hydrolyze GTP, that energy is used to remove the elongation factor from the situation, and now we've exposed the amino acids next to each other that we're about to form a peptide bond between. So that's step one in the elongation phase, bringing in and verifying the next charged tRNA molecule. Now step number two is actually making a covalent bond between them. So here's what we're doing, and if you want to think about the previous figure, they're both again showing the same thing. I've lined up my amino acids, and what I'm going to do is form a covalent bond between what was the last link in the chain and what is now the last link in the chain. It is important to recognize, remember, that covalent bonds require an input of energy. This is an energy-dependent process. It's endergonic. Who's paying for this? Remember, unless you're told otherwise, ATP is our go-to energy currency. But in that figure, do you see any ATPs, any little starbursts or anything showing energy? No. So who's paying for this? If we break a covalent bond, perhaps that covalent bond has enough energy in it to pay for the one we're about to make, and that's exactly what we're going to do. We're going to break the bond. Notice that in this side of the figure, there's a solid line between that amino acid and that tRNA. That indicates the covalent bond that was formed during the charging process. And I mentioned that I wanted you to tuck that away. That interaction didn't have to be covalent, but it used a molecule of ATP back when it got charged, so that we didn't have to use a molecule of ATP inside an already crowded ribosome. So we're just taking advantage of work that we've already done. You break that bond between the amino acid and its tRNA, that releases enough energy to form the covalent bond between the amino acid that just got in there and what was already there. That is peptide bond formation. And as any good enzyme, that activity has a name that ends in ASE. That's called peptidyl transferase activity. But here's the twist. The peptidyl transferase activity in a ribosome is not in any of the polypeptide components. It's in the biggest of the rRNA components, the 28s rRNA in eukaryotes, the 23s rRNA component in a bacterial large ribosome subunit. In other words, a ribosome is a ribozyme. This is a structure that can catalyze a chemical reaction, peptide bond formation, through the three-dimensional structure in an active site by a folded RNA molecule. So it's a ribozyme. That's peptidyl transferase. Now, we would like to add the next one, but remember, every next tRNA has to come in the A-site, and right now my A-site's full. So what I would like to do is shift everything over, exposing the next codon in a now open A site. And that's exactly what translocation is. The shifting of the ribosome relative to the mRNA, three bases closer to the three prime end. So here's where I am at the end of step two. Now what I do is I just shift everything over. It's relative movement. It doesn't really matter whether the mRNA is being pulled or the ribosome is doing the sliding. Both examples can be found. The bottom line is I expose the next three bases in the A site, and I'm ready to do it all over again. This also requires help, and it's also another elongation factor. It's not the same one we used before. In E. coli, this one's called EFG, and in eukaryotes, it's called EF2. Again, they're not the exact same molecules, but they do the same thing. So these elongation factors play the same roles in every organism. And in the translocation phase, we're going to try to shift that ribosome, and we're going to need energy to do that. Moving a house is an energy-intensive process, and that's what we're doing. We're moving the house. And you'll notice that it looks like the tRNAs got up and went to a different room. That's not the right metaphor to consider. The tRNAs didn't get up and move to another room. They blinked, and the house shifted, and they now find themselves in a different room than they were in a moment ago. So what was in the A site now is like, whoa, what just happened? I'm in a different physical space. Now the tRNA that is at the moment carrying all of my protein now finds itself in the P site, and I've got an open A site ready to do it all over again. And that's exactly what happens. Let's walk through another round of elongation. So this is my result at the end of translocation. I'm ready to do step one, binding of the next charge tRNA. Don't forget that this charge tRNA doesn't come in by itself. It's got an elongation factor attached to it with a GTP. If codon and anticodon pair, then the GTP is hydrolyzed. The elongation factor is peeled away, popping the amino acid back up into a position where it's now ready to be put right next to what used to be the last link in the chain. Step two, peptide bond formation through the RNA component of the subunit. I'm gonna get energy to form that covalent bond right there between this, it's glutamic acid and arginine. Doesn't matter what the identity of the amino acids are. The peptide bond is the same no matter what. This peptide bond, this covalent bond, is paid for by hydrolyzing that covalent bond right there, which again was the product of the charging reaction. So we allow the molecule to bring in its own energy source so that we don't have to get so crowded on the inside of a ribosome. Then once I've extended my chain by an amino acid, I would like to do it all over again, but right now I can't because my A-site is occupied. So I'm going to shift over three more bases, and now I'm finally going to kick out this tRNA that has been uncharged for a while, but didn't have anybody sort of forcing it out. Now it gets forced out, and the next uncharged tRNA finds itself in the E-site. And when the next round of elongation happens, then the red uncharged tRNA molecule will get kicked out as well. So translocation always opens up the next A site, and this is going to continue. These three steps, binding of the next charged tRNA, peptide bond formation, and translocation, those three steps are going to repeat exactly the same way over and over until one of the three stop codons enters the A site. One of the biggest misconceptions in this class is thinking that there's a tRNA that has an anticodon that recognizes a stop codon. There is no such thing. There is no tRNA that has an anticodon that recognizes a stop codon. We don't use tRNAs. There's nothing tRNA-charged-with-a-stop thing. That doesn't exist. Instead, we're going to use another helper protein, and again, we're going to call it a factor. This time, we're going to call it a release factor. There are different numbers of release factors in different domains of life. Don't worry about how many there are. Again, they all do the same thing. A release factor is a protein. A protein that physically mimics the shape of a charged tRNA molecule. It's like a chameleon, right? Or a shapeshifter, whatever. It physically resembles the shape of a charged tRNA molecule. It's going to get in there and trick peptidyl transferase into trying to do its job. It's not going to be able to because there's no amino acid that's going to pop into that active site. So the ribosome is just going to say, okay, you fooled me. Let's let everything go. Let me show you how that happens. So here, this purple blob looks like another tRNA, doesn't it? It's not. It's a protein called a release factor. How do you know that this is not a nucleic acid? If it was a nucleic acid, we'd probably tell you what the nucleotides are. There's no anticodon in a release factor, because an anticodon is a set of three bases that forms complementary base pairs with a codon. A release factor is a protein. But it does physically resemble a charge tRNA molecule. So as soon as it gets in there, the ribosome thinks, oh, I'm ready to do step two. We must have just finished step one, bringing in the next charge tRNA. It's looking exactly like what it's supposed to find. So what does it do? It breaks the covalent bond between the tRNA and the growing polypeptide chain in the P site, then attempts to move it over and covalently link it to the amino acid that just came in the A site, except that there is no free amino acid that just came into the A site. So I've already broken the protein off of the ribosome. There's no going back. So I just let go of the polypeptide chain. The release factor binds to a stop codon. It does not hybridize to a stop codon. It binds to a stop codon because it wasn't really a U-A-G. It's very tempting to think of these genetic sequences as actual letters. They're not actual letters. It's a chemical structure that's part of a longer molecule called an mRNA. And that structure has a three-dimensional shape. And that shape is compatible with this protein called a release factor. So that release factor binds that mRNA and tricks the ribosome into breaking the bond. Between the tRNA that's in the P site, notice that this is showing the breaking of that bond right there, and I'm going to attempt to move this over and add it on to the amino acid that just came in the A site, but there is no amino acid that just came in the A site. So the ribosome has no choice but to let go of everything. And once you've let go of everything, the subunits themselves just dissociate and go looking for new work. If this was a 40S subunit, it's going to go looking for the next 5' cap. If it's a 30S subunit, it's going to go looking for the next Shinedalgarno sequence. And we do it all over again. That's how we terminate translation. So subunits separate, then you release the release factor and the mRNA, and if you want to make more protein from that mRNA, you need to go back to the beginning, which we have the capability of doing. Usually we want to make a lot of protein from It took a lot of time and resources to do this transcription and 5' cap addition and intron removal and exon joining, maybe alternative splicing, poly-A tail addition, export out into cytoplasm. This was not a brief or inexpensive process, so we'd like to be able to make as much protein from that as possible. Now I get to revisit one of my favorite questions, when do things happen in biology? As soon as they can. So if you have a ribosome that has initiated on, let's say, a 5' cap and scanned down until it found an AUG, as soon as you begin the elongation phase and the ribosome moves its way down the line, is there any reason why we couldn't do it all over again with another ribosome? You don't have to wait a ribosome, a fresh ribosome, doesn't have to wait until this first ribosome finishes before it begins. Things happen as soon as they can. So as soon as the ribosome has moved far enough away to get an initiation complex loaded in that place, an initiation complex is going to get loaded into that place. And you begin this assembly line production of protein that's either called a polysome or a polyribosome. You can have as many ribosomes on an mRNA as you have room for. As soon as you move far enough down the line to get an initiation complex, you're going to get the next initiation complex. So if we take a look at what this looks like in cartoons, and just to demonstrate that this is not made up, this is real here, the horizontal-ish line here is the mRNA. The blobs are ribosomes, and so the gold line is the mRNA, the orange blobs are the ribosomes, and the green lines are the polypeptide coming out of those ribosomes. So you see the string coming out of these beads here? That's the growing polypeptide chain. Why are the strings longer as you make your way down the line? Because they've been on there longer and had more time to carry out translation. The initiation, whether, notice this figure doesn't tell you whether I found an AUG by finding a 5' cap or a ribosome binding site. Because it doesn't matter. Polyzone formation can happen in any living organism. Polyzone and polycistronic are not the same thing. Polycistronic means I can have multiple proteins made from a single mRNA molecule. Polyzone means I can take a coding region and be making multiple copies of it simultaneously as long as one ribosome moves far enough down the line to get the next one in there. So why are the lines shorter towards the 5' end? Because these just initiated. They haven't had a lot of time to elongate their protein chain. By the time I get down here, near a presumed stop codon, these polypeptides are quite larger. And so the longer lines indicate the closer you are to a stop codon. Alright, so remember, polyzone formation can happen in any living organism. I'm
