 Thank you. I was almost late to class today. I was reading this book earlier about anti-gravity. I couldn't put it down. Thank you. Good afternoon, everyone. Happy Monday to you all. Welcome to MCB 150. On Friday, we stopped just after getting started about the process of DNA replication. We sort of introduced it. Let me do a very quick review. We introduced it by being a series of steps. We need to figure out where to start. We call that the origin of replication. Then we're going to need to separate the strands. That's kind of where we're going to pick up today. Then we're going to need to get things started. We'll call that priming the pump. Then we're going to do the bulk of the DNA synthesis. And then the cleanup crew is going to need to come in and finish up the job. So again, that's sort of our outline for what we started with and what we are going to continue today. The one part that we didn't get to in terms of origins of replication is contrasting the single ori in a typical bacterial chromosome with multiple oris in eukaryotic chromosomes. Here's the deal. E. coli has a genome of about four and a half million base pairs, and it can knock out all of its DNA replication starting from one spot in about 20 minutes under good growth conditions. That's pretty amazing. It can even do DNA replication as a process faster than eukaryotes. If, I'm saying if humans could replicate DNA as quickly as E. coli, and by the way, we cannot, even if we could, we have so much more DNA in our cells than E. coli has in its cell that it would take us two weeks. If we started in one place and went the same pace as E. coli, it would take two weeks, not 20 minutes, and we don't have that kind of time. So the answer to that problem is start in more than one place. Eukaryotic genomes, which tend to be larger, tend to have more than one origin of replication. So you see these replication bubbles that are getting larger and closer to each other. And you can see the sort of time-lapse cartoon imagery here. We start in different places, there are three oris represented in this strand, each one of them creates an independent bubble, and you notice the direction, the little red arrows show the direction of the replication forks. There is a process when they run into each other that they just sort of smooth out and become two strands. So now you've got two identical double helical molecules here, and even under electron microscopy you can see there's a replication bubble here, the little pink arrows indicate the forks, there's another bubble there, and then there's a bigger bubble here. So they immediately meet, and once they merge, you've got separate strands of DNA. So that's how we can do DNA replication in a reasonable amount of time. Now I'm not saying that a human cell can replicate six billion base pairs of DNA in 20 minutes, it takes about eight hours, but still, at least that's a workable number. In two weeks we can't wait on it. Eight hours, as long as other things are continuing to go on, we can live with that. So, that was our introduction to step one, figuring out where to begin, and in this case, how many places to begin. Step two was separating the strands. The good news is, there's a molecule for that. The bad news is, there's more to it than just separating the strands. The molecule that does it is called helicase. Helicase is the yellowish molecule in roughly the center of this cartoon figure right here. Helicase is an ATP-driven plow, if you will. It plows the road, it pulls the strands apart. Now remember that every base pair is held together by two or three hydrogen bonds. Any one base pair is not going to put up much of a resistance, but a whole strand of nucleic acid, that's a lot of hydrogen bonds. And it's not going to be easy to just start pulling. So helicases use ATP as an energy source to peel those strands apart. Two things about that. What's happening ahead of you as you start to pull those strands apart is that you start twisting and tightening the DNA in front of you. And you would ultimately get to a point where you just can't pull anymore because everything's been twisted too much. So there's an enzyme called a topoisomerase. Topoisomerase. That is out ahead of the replication fork. Important distinction. A topoisomerase is not in the replication fork. Helicase is basically pushing the fork forward. But the topoisomerase is out ahead of that, relaxing the DNA. We'll talk about supercoiling on Wednesday. Relaxing the DNA to a point that helicase can actually keep separating. But here's the other problem, if you will. If I take double-stranded DNA and I separate it and I let go, what do you think is going to happen? What would your prediction be? It's just gonna go back together, it's gonna wind back around itself. So we need to keep it single-stranded at least long enough to get machinery in there and start using the single strands as templates. So you can see this figure which is talking about separating the strands shows more than just helicase. Yes, helicase is the one doing the separating, and it's using ATP as an energy source to do it. But topoisomerase is out ahead of the fork making sure that the DNA doesn't get bound too tightly, and once helicase has pulled some strands apart, these proteins which are conveniently called single-strand DNA binding proteins talk about being obvious with your naming, that's what they do. They bind single-stranded DNA and they keep it from becoming double-stranded again until you can get your machinery in there. So that's separating the strands. Then we have to make DNA. I'm going to reintroduce you to the general idea of phosphodiester bond formation. So, oh, oh, but this is going to happen in two separate steps, and it's important to recognize which step comes first and why. When I'm going to form a phosphodiester bond, the first thing that I do is make sure that it's the right nucleotide in the first place. Remember, you're going to come in with three phosphates, and I'm going to ultimately chop two of them off. I don't want to waste that nucleotide. If there's a T in the template and I'm supposed to put an adenine opposite that, I don't want to bring in by accident or just by random a DGTP, break two of its phosphates off, and then try to line it up with that thymine and say, oh wait, this doesn't fit. Then I just burned the energy for no reason in that DGTP molecule that I brought in. So before I break the phosphate groups, I'm going to let it try to line up and form complementary base pairs, complementary bonding, a hydrogen bonding. And if it does, well, first of all, if it doesn't, and I got a three in four chance of it not being the right one, well, then I just kick it right back out and I don't waste its energy. But if the one that randomly came into the active site is the right one and it lines up against the template, then and only then do I break the pyrophosphate group off of the incoming nucleotide and use that energy to form the new phosphodiester bond. So as a reminder, this part of the figure, or this figure right here, or a version of this, we've already seen. We saw this when we did our very, very brief introduction to nucleic acids when we were talking about macromolecules. Over on the right is our template strand. And notice that it's upside down, if you will, the way it's been drawn. That's to remind you that the strands are anti-parallel to each other. This is the strand over on the left that I'm trying to lengthen. You always, always, always add the next nucleotide to the 3' hydroxyl of what was already there. So we're going to be adding the next nucleotide to the lower left of this figure at that hydroxyl that's been highlighted in blue. This part is not new. I do want to point out that because we have used helicase to peel the template strands apart and single-strand DNA binding protein to hold them apart, I now have access to this information, and I can see that the next base in the template is a cytosine. Any nucleotide has the right to come in and try their hand at matching up here. If it's a thiamine that comes in, well, T's and C's don't base pair, so it's just gonna get kicked back out. But if I bring in the appropriate base, this time a guanine, a DGTP, if I bring in the right nucleotide, the very first thing I'm gonna do is make sure that I can form the three hydrogen bonds that you see formed on the right. Because I don't want to waste the energy in this molecule by just assuming that it's right, breaking the pyrophosphate group off, and then saying, uh-oh, you don't actually fit here. So let it prove that it fits. Then break off the two phosphates, use the remaining phosphate on the DGTP, although now it would be a DGM key, and you can see by the dotted line or the dotted circle here, this is where our dehydration synthesis reaction is going to occur. There's the phosphodiester bond. We broke our pyrophosphate to release more energy, and now those inorganic phosphates can go be phosphates for some kinase that might need a phosphate for something. So I get asked a lot what happens to the phosphates that are left over. There's never going to be a long wait time before that phosphate is going to be used by something in the cell. So everybody that's a byproduct for this is a substrate for somebody else. Okay, that's what we want to do, and we want to do a lot of it. But we have a couple of really big problems to get started that we have to solve. Here's our first problem. DNA replication has to be bidirectional. So I want to show you what I'm talking about here. If the blue strands represent parental DNA, and the circles on those strands represent the origin of replications, then I want to synthesize DNA from both template molecules on both sides of the ori. That's four different regions of DNA that I want to be synthesizing, right? I want to be starting here and going that way. I want to be starting here and going that way. I want to be starting here and moving that way. And then fourth quadrant, I want to be moving from the lower middle to the lower right. I want to be able to do all of those things. But you cannot violate the rule that nucleic acid synthesis must, must, must occur in the 5'3' direction. I am very careful in my use of absolute terminologies. I'm using it here. It's the only way to synthesize nucleic acid. It has to be done in the 5'3' direction. So let's remind ourselves about the directionality of the various lines that have been drawn on this figure and what problems that might create. I can randomly and arbitrarily, it really doesn't make any difference how I begin labeling things because I haven't identified what end is what of anything yet. So let's arbitrarily say that the top blue strand over on your left is the 5' end. Then that means that over on the right of that top blue strand is the 3' end. Since the blue strands are anti-parallel to each other, that means the lower left of the bottom blue strand is the 3' end and the lower right is the 5' end. Just anti-parallel molecules. Now remember that anything we're making, anything, is going to be anti-parallel to the template that it was synthesized from. So if the top strand, the top blue strand, is 5' on the left and 3' on the right, then that means this purple fragment has to be 3' on the left and 5' on the right. And the one right next to it, you don't change the directionality just because you're on the other side of the ori. It's still ultimately the same template molecule, and since that template molecule, the top blue strand, has 5' on the left and 3' on the right, that means this upper right purple molecule is also 3' on the left and 5' on the right. So it's important that you know that you should be able to fill in all of those numbers just based on a point of reference. If I tell you the top strand is 5' on the left, then everything else fills itself in. Now that doesn't in itself show you the problem. Let's take a look at the problem. Remember, we have to be reading our template in the 3' to 5' direction so that we can be synthesizing an anti-parallel molecule in the 5' to 3' direction, the only way that it'll work. So if I wanted to start at the ori, oh, and by the way, that's the rule. You have to start at the ori. It is the origin of replication. If I want to start this upper left quadrant purple strand right here, I would be working from the 5' to the 3' direction, and that's okay. So it gets the green checkmark, right? Nothing wrong there. But look at what I need to do in the upper right. I need to be able to make an anti-parallel and complementary molecule with my overall direction of movement. There's a difference here. You have to be careful in understanding the terminology. When I talk about the directionality of nucleic acid synthesis, that's possibly different than the movement of the machinery. The machinery, if you're starting at the ori, half of your machinery goes that way and the other half of your machinery goes this way. So I need to be overall moving in this direction, but I have to be making my new molecules in the 5' to 3' direction. And if I were to just start at the beginning, start at the ori, and work my way left to right, because that's the overall direction I want to move, I would be, theoretically, synthesizing DNA in the 3' to 5' direction, and that cannot happen. So there's the problem. The problem is, I could do this lower right quadrant, start here, move that way, 5' to 3', that's fine. But I have the same problem in that third quadrant. So part of my DNA doesn't seem to be able to be replicated the way I need it to. How are we going to fix that? We're going to fix that... by doing what's called discontinuous DNA synthesis. We are going to start at the ori, and where one of our strands, we can start at the ori, we can move in the appropriate 5'3'' direction, and as long as you continue to plow the road for me, I can continue to get in there and do work. As long as Helicase keeps doing its job, I can keep adding in nucleotides. And that uninterrupted synthesis is called the leading strand. But what I'm going to do on the other side, I'm going to have to go back and work my way in. So let me give you a visual of this. Let's say that my job is to lay in tiles or bricks or something from that wall to that wall. And I can put them in, but I have two limitations that I cannot get around. One of them is, I have to start in the middle of the room, and the other is that every tile, brick, whatever that gets laid down, has to be done from your left to your right. Those are the rules I have to operate under. Well, okay, I can start right here in the middle. Remember, I have to start the overall process in the middle of the room. Well, okay, I need to go to that wall, just lay in brick, brick, brick, brick. This is leading strand synthesis. This is easy. All I have to do is wait until I get to the end and I'm done. The problem is the other half of the room. I have to be laying in bricks facing this way and laying them in in this direction, but I have to start here. How could I do that? Don't think like a cell for a second. Think like an engineer. How could you orchestrate me to do what you're the director, I'm the actor. What would you want me to do? How could I put, what do you think? Move me over here and let me then brick, brick, brick, brick, brick, brick, brick. Now, the question is, why don't you go all the way over to the end? It hasn't been unwound yet. We're gonna unwind a little bit at a time. There's a helicase can only go so fast. Plus, you don't want your DNA exposed for that long. So, unwind a little bit, expose a little bit, then position me here so that I can go back to the middle. Then, in the meantime, you continue to unwind more. You just move me farther back here, and then I move until I run into what I did the first time, and then you move me back here, and I run in the right direction, I run into what I did the first time. That's called lagging strand synthesis, and those pieces that were made discontinuously are called Okazaki fragments, after the Japanese scientists that discovered this lagging strand synthesis ability. That way, I'm obeying all the rules. I'm starting in the middle of the room, but I'm always moving from your left to your right. Or, to use more scientific terminology, I'm always starting at the ori, but all of my actual synthesis is 5' to 5''. Got a couple of pictures that show this. Here, we've just started separating the strands. We see the ori is labeled for us. I can get my machinery in there and begin my leading strands on both sides. They're sort of mirror images of each other. So, what's happening in the upper left is also happening in the lower right, and vice versa. I can start by initiating at the ori, and as long as you keep opening up more DNA, I can continue to extend my leading strand. That's no problem. Once I've unwound enough DNA, once helicase has made, and there's a helicase moving this way, and a helicase moving this way, once the bubble has gotten big enough, now I can start to load in the machinery, going back here, load in the machinery, and work back towards the middle. So, now, in the upper right and lower left, you see your first Okazaki fragment being laid in, once enough DNA has been unwound. Then we continue that. You see, as I'm making those first Okazaki fragments, they head back until they run into the beginning of the leading strand. In the meantime, I've been unwinding more DNA. Now I have room to load in the next Okazaki fragment. And while it's being synthesized, I'm continuing to unwind more DNA, giving myself room to load in machinery and do the next Okazaki fragment. That's called discontinuous DNA synthesis. And so, the whole of DNA replication is called semi-discontinuous, because the leading strand is called continuous synthesis. Lagging strands are called discontinuous. So, the overall replication of an entire molecule of DNA is called semi-discontinuous, because it's partially continuous and partially not. That solves one problem, but it doesn't solve the second problem. The very tools that I depend on to do the bulk of my DNA synthesis can't start a strand from scratch. That's an odd problem, isn't it? So, I want to show you what they can't do by first illustrating what they must have. They have to have a free 3' hydroxyl to add the next nucleotide to. Now, if those are the parameters that we're working under, you can see how we can't add the first one. If you are absolutely required to bring in the next... Already, we have a language problem with this, don't we? Where's the first one? If you have to begin the sentence with, every next nucleotide has to be added to the hydroxyl group at the 3' end of what was already there, then what if there wasn't anything already there? Something has to be able to get us started. So, here, there we go. This seems simple. I've got single-stranded DNA. Well, no, we don't have to assume it's DNA. It's got thymines in it. So, this is a single-stranded template of DNA. There's the 3' end. There's the 5' end. All I want to do is let an adenine hybridize. 全我想做的是讓一種鋏鋏合成。 That's a fancy word for forming complementary base pairs, 這是一個好好的說法是為了組成相同的基礎組合。 Allowing those hydrogen bonds to form between an A and a T, 讓那些氧化合成成為A和T, or between a G and a C, that's called hybridization. 或是G和C,那叫做合成。 So all I want to do is hybridize that adenine opposite that thymine. 全我想做的是合成那鋏鋏, opposite that thymine. 相反的鋏鋏。 But I can't. DNA polymerases, DNA synthesizing enzymes, 但是我無法做到。 DNA polymerases, DNA synthesizing enzymes, can't do that because they don't have a free 3' hydroxyl group to add. 而DNA 酸化酸素不可以做,因為它們沒有一個充分的3'酸化酸組合。 There was nothing here. There was nothing for it to be added to. 這裡沒有東西可以加進去。 So let's take the bottom molecule, put it at the top over on the right. 現在我們把底部的物質放在右邊。 Now if this is what we have to start with, I've already got a nucleotide in place. 現在我們要開始做的是這個。 I've already got a nucleotide in place. 已經有一個鋏鋏。 It honestly doesn't matter what kind of nucleotide it is. 實際上,它不重要是哪種鋏鋏。 It doesn't matter whether it's DNA or RNA. It doesn't matter whether it's ACGT or U. 無論是DNA還是RNA,無論是ACGT還是U。 It doesn't matter whether there's one nucleotide or a dozen. 無論是一顆或一千。 I have something to add on to, and I can do this. If I'm a DNA synthesizing enzyme, I can add base number two to base number one. I just can't add base number one. It is a limitation that I cannot overcome as a DNA synthesizing enzyme. That's pretty bad news. It's sort of almost a literal non-starter, isn't it? Well, there's a solution to this. RNA synthesizing enzymes called RNA polymerases, they don't have that limitation. They can actually start strands from scratch. They can do the left-hand side of the previous slide. They just say, oh, except they wouldn't be using thymines, they'd be using uracils. There's an adenine in a DNA molecule, I'll just let a uracil form a couple of hydrogen bonds with it. No skin off my nose. And RNA synthesizing enzymes can do that. The molecule that does this in E. coli, anyway, every organism has this problem. Different labs study bacteria, some labs study eukaryotes, and when different labs study different things, they come to different conclusions on what they want to name them. Anytime I give you names of molecules with respect to DNA replication, they're going to be E. coli names. Those are the ones we've known about the longest. There are fewer of them to talk about. Eukaryotes do everything that the bacteria does, and then some. So the list gets a lot longer. We're going to stick to these relatively simple E. coli names. The name of the RNA synthesizing enzyme that can start a new strand from scratch in DNA replication is called primase. And primase makes primers. You could, theoretically, just put in one nucleotide and get out of the way. That would give you the only thing that the DNA polymerases require. That free 3-prime hydroxyl group. That would be, theoretically, all we need. Remember, I said it doesn't matter whether it's RNA or DNA. It doesn't matter whether there's one of them or 12 of them. It doesn't matter what the sequence is. All I need is that hydroxyl group. So why would I put in, it says 5 to 10 here. Honestly, I think it's more like a dozen most of the time. Why do we keep going to 12 instead of stopping at 1 or 2? The answer is stability. If I allow primase to get in there, put in just one base, pull it out of the way, by the time I get a DNA polymerase in there to pick up where it left off, if all that's holding that base pair together is two hydrogen bonds, it's going to be long gone by the time I get the DNA polymerase in there. So I make my primer long enough that it's not going to go anywhere. But I don't make it excessively long. If 12 is good, why is it 25 better? Because I'm going to have to remove those RNA bases at some point. And the more I put in, the more I have to take out. So by making my primer excessively long, sure, it's definitely not going anywhere. But I'm giving my cleanup crew more work to do. There's a little teaser at what the cleanup crew is going to have to do. They're the ones that are going to have to get those RNA molecules out and somehow replace them with fresh DNA bases. And there's a good way for them to do that. Now we've got what we need. Whether your primer, which is a sequence of RNA nucleotides, whether your primer has 5 nucleotides, 10 nucleotides, 13 nucleotides, not the point. And I want to also say for the third time, it's not that these are RNA molecules that's important. It's just that the tool we have at our disposal to do the job of giving us the free 3 nucleotides is not going to be enough. So we're going to have to do that. And we're going to have to do that. prime carbon is in vitro, not in vivo. In the lab, when we're priming DNA synthesis artificially, we need to put in primers ourselves. We don't put in fresh molecules of primase. We make our own artificial primers, and they're DNA primers, illustrating the fact that it's not the ribose that's important in this equation. It's the hydroxyl group at the 3 prime carbon.
