
 Thank you. Are there any pre-optometry students in class today? Anybody on a pre-optometry track? Okay, so there's at least one. I've got a question for you. How many optometrists does it take to change a light bulb? Is it one or two? One or two? Again, one or two? Good afternoon, everyone. Happy Friday to you all. Welcome to MCB 150. I haven't had any class announcements for a while. I have an important one today. I'm changing something in the schedule of what's happening later in the semester. Specifically, I'm changing the schedule of events for our exam weeks that are coming up. We have two more Thursday night exams, the next of which, by the way, is two weeks from last night. So it's around the corner. And then there's one a month from that. And then finals are going to be unaffected by what I'm about to say because finals week is its own thing. And during exam one week, from this point forward, the day before the exam is going to be a regular official class period. And the day after the exam is completely free, completely off. All right? Now, I'm going to clarify a couple more things because you might be wondering. The material that we cover on that Wednesday lecture, I don't expect you to have actually learned it and internalized it in 24 hours. So that will not be the same material. It will be the beginning of the next unit's material. So the Wednesday class period will be official. You should be here. There will be learning catalytics questions and so forth. But that material begins the next unit. It's not going to be on tomorrow night's exam for those two Wednesdays. And in case you're worried too much about losing that optional review session on the Wednesdays, later this week, I'm going to be on the next one. And at the end of that day, I have my regular student office hours anyway. So plenty of opportunity for you along the way, any time between now and in that afternoon, to get your questions answered. I'm doing this for two reasons. Number one, because the day after exam three, I'm going to be in Phoenix at a professional conference. So it kind of necessitated a change. And then I was thinking about the fact that the day after exam two happens to be the drop deadline. It's also, in case you haven't noticed, and I'm sure you have, the day before spring break starts. So a lot of you are probably thinking, oh, I'm going to be there anyway. Now you don't have to worry about missing anything. But I'm going to make, I'm not taking the day off. I'm going to make myself available basically the whole day from the beginning of my Friday Zoom student hours all the way through the end of what lecture would have been. I'm going to be hanging out in that Zoom space, waiting for people to come by to ask questions about the fact that it is the drop deadline. You will have an answer key posted by then, earlier than usual. You'll be able to see roughly how you did on that exam the night before. And some of you are going to have questions about, should I stay or should I go, with respect to that being the drop deadline. Please don't make any rash judgments. We're only approaching the halfway point in the semester. It's best to talk with somebody about what you've been doing so far and where you might be able to make changes in the second half of the semester. Don't just automatically assume that because you had a rough exam the night before, oh, I need to drop the class. Please talk to either your instructor and or an academic advisor. And oh, by the way, I'm both of those things. I'm an instructor and an academic advisor. So please feel free to talk with me. I'll re-announce that as we get closer. I'm just sort of putting it on your radar right now. Those are the announcements. Pretty big changes, but I think in everyone's best interest. Now, I want to go back to our central dogma figure. And if, in the end, all of what we're using our genetic information for is in the ultimate production of protein, then why the extra step? Why do we need to go from DNA to RNA to protein instead of going directly from DNA to protein? Well, it's because we have, despite the fact that we have very large genomes, we do have a limited number of copies of a given gene, of a given unit of information. In prokaryotes, they've got one. In eukaryotes, diploid. Well, I shouldn't say eukaryotes. In diploid organisms, maybe you've got one copy that you inherited from your mother and one that you inherited from your father. That's still not very many copies. And if the ribosomes, if you needed to make 500,000 copies of a protein, which is not an unusual number, if you had to make half a million copies of a protein and your ribosomes had to wait in line for one of the two copies of the gene to be open and accessible, you'd never get the appropriate number of proteins made in a relevant amount of time. So the answer to what this intermediate step gives us is amplification. If I could make 100 copies of RNA identical, and each of those 100 could be translated by 100 different ribosomes, all of a sudden, I can, in relatively short order, make 10,000 different copies of a protein. Whereas if I were waiting, if it was just a one-step process, I might only be able to make 100 copies of that protein in that same relative amount of time. So we can really amplify our signal available to us by having the intermediate step. It seems like extra work. It's actually way more beneficial than the extra work. And the process, this is just a reminder, the process of taking the information in a DNA molecule and turning it into the information in an RNA molecule is called transcription. So I'm sure you've heard of things like mRNAs and RNAs, and we'll talk about them in time. I want to really solidify the point that transcription is not necessarily making a protein-coding RNA. It's not just making your messenger RNAs. Transcription is, quite simply, the process of making an RNA molecule from a DNA template. That's the process of transcription. So let's talk about how E. coli does transcription. Well, we're going to be a little more general at first, and then we'll talk the specifics for the same reasons that we talked about E. coli specifics in DNA replication. They're more simple, they're largely the same, and that way we only have to memorize, at the moment, one set of names. In general, but this slide is really more general, what are the tools that carry out transcription? They are specific molecules called RNA polymerases. And to go back to our terminology from a few days ago, X-dependent, Y-synthesizing, where X is what are you reading as your template to identify what's the next base I'm supposed to add, and Y is what am I making right now. An RNA polymerase is a DNA-dependent RNA-synthesizing enzyme. It's not the first one of those kind we've seen. Primase is also a DNA-dependent RNA-synthesizing enzyme. The primase works in the process of DNA replication. RNA polymerases work in the process of transcription. At their heart, they serve the same role, they just are different molecules that are working in different cellular processes. Transcription is going to occur in a set of steps that's going to sound somewhat largely familiar to a set of steps that we've already talked about. Well, we have to learn where to start, we have to get things going, we have to extend what we've started, and then we need to wrap things up. Now, that process kind of sounds familiar, doesn't it? It's more or less the order of events for DNA replication. We threw a few wrinkles in there, like we called it priming the pump, and there was a clean-up crew that was involved. But for the most part, with any cellular process, you have to be told where to start, you have to get things going, and that process is almost always called initiation. Then you have to keep things going, and that process is often called elongation. And then you need to be told where to stop, or in the case of DNA replication, you just go until there's more work to do. There's really no stop signal, if you will, for DNA replication. You're not done until you've copied the very last base. But for every other one of these processes that involves genetics, there is a discrete stop sign as well. So yes, let's talk about the E. coli molecules, and we'll start, and today will all be about how bacteria do transcription. Then we'll throw the, oh by the way, these are the differences between bacteria and eukaryotes when we come back on Monday. But today it's going to be all E. coli stuff. The E. coli, and essentially all prokaryotes, have a single type of RNA polymerase. And it's got six different subunits to it. There are two identical alpha subunits, and by subunits remember I mean individual polypeptides, that are components of a larger functional protein. Two identical alpha subunits, a beta subunit, a closely related but not identical beta prime subunit, an omega subunit up on the top, and a sigma subunit. When all six of those are present and functioning, that RNA polymerase is given a special extension on its name. It's called the RNA polymerase holoenzyme. Holo means all of the parts are accounted for. It's the whole thing. So the E. coli RNA polymerase holoenzyme is all six of those subunits. Now here's where it gets a little interesting. The sigma subunit is detachable. And if you change the shape, you're going to change the function. If you change the chemistry, you change the biology, right? If the sigma subunit is all of a sudden not there anymore, it changes things, and it should not come as a surprise that the overall enzyme is going to behave differently. Since it behaves differently, we created a new word, or new phrase, to describe the RNA polymerase without the sigma subunit. That's called the core enzyme. So if you see RNA polymerase holoenzyme, you know that the sigma subunit is still attached. If you see RNA polymerase core enzyme, you know that the sigma subunit has been displaced and can rejoin a core enzyme some other time. There's lots of copies of holoenzyme floating around in an E. coli cell. A lot of them are now core enzymes because their sigma subunits have detached. There isn't... There's no monogamy in the world of forming a holoenzyme. Any sigma subunit can attach to any core enzyme. It's not waiting around the cell looking, where are you, where are you? The one that it was attached to a second ago. They're all built the same anyway. All those alpha subunits, those beta primes, those sigma subunits, the sigma subunits are different, but that's beyond the scope of MCB150. The rest of the core enzyme, it's all built the same, so any sigma subunit can attach to any core enzyme, creating the next holoenzyme. And unlike DNA replication, where as I reminded us a second ago, you go until you're done, you copy every single base in that organism's genome, in transcription, you copy discrete units of information. We're going to call those genes. So transcription is the production of RNA from a gene in the DNA. And in bacteria... Most of the DNA consists of genes. There isn't a lot of non-coding DNA in a bacterial cell, but that doesn't mean you want to be making all of your RNAs all the time. So transcription is also the process of figuring out which ones of these genes do I need to be making right now, leaving the others alone. As opposed again, by contrast to DNA replication, where you find an ori and you just go until there's no more DNA that needs to be copied. Transcription is way different. Eukaryotes are similar. Eukaryotes have more DNA in our genomes that don't code for RNA molecules, but we still have the same ultimate issue that we only want to be transcribing the genes. The stuff that isn't even a gene, doesn't even get turned into an RNA. An RNA polymerase is never supposed to be interested in that. So we look for those regions of useful information. There needs to be a landmark. There needs to be something there, like an origin of replication did in the world of DNA replication. In that conversation, last time, when we were talking about replication, the start sign was a sequence of bases that we called an ori. Now, the sequence of bases, which is going to be different, but equally important, is called a promoter. So a promoter is, in transcription, what an ori is in DNA replication. It's the sequence of bases that creates a three-dimensional landscape that someone is responsible for recognizing and saying, oh, hey, I found one. I found a gene. Let's transcribe here until we're pulled out of the way, until we're terminated. So, let's start by talking about promoter sequences. We know their sequences of bases. We know they should be similar. If I've only got one kind of RNA polymerase, if I need coli, I've only got one kind of RNA polymerase, then all of my promoters should look pretty similar, right? That doesn't mean they all have to be identical, but they are gonna be surprisingly similar, at least in certain areas. All right, so if this represents, with a figure label here that says transcription unit, we can call that a gene. We're gonna go from promoter to terminator. And the promoter, with one exception that we won't see today, promoters are ahead of the beginning of transcription. In genetics, we don't say ahead, we say upstream. Downstream means in the direction of the gene itself. So the promoter would be upstream and if you aim yourself to the left on this figure you're headed upstream if you orient yourself Going to the right on this figure the way it's been drawn then you are headed downstream with one notable exception That won't come up today promoters are upstream of the genes that they control. So we've got a promoter region Here's where we're gonna start transcription. There's where we're gonna stop transcription. We're gonna zoom in on that green region So now we resume it on this the parts in green are Could be a hundred or more nucleotides or base pairs in length. A promoter region can be fairly long. It's got, in genetics we call it a footprint, the DNA footprint. It's how much DNA is covered by the molecule that's responsible for binding to it. And the footprint of an RNA polymerase in E. coli is about 120 base pairs. So it's gonna cover a lot of territory, but of those 120 base pairs, only six of them over here and six of them over there are actually important. I call them hot spots. That's not technical terminology and don't think that means temperature of some kind. There's just two really important, super important regions within this otherwise important region that we're calling a promoter. So you'll notice that there are two hot spots here that have sequences identified. Notice the positioning of them. One of them in... Oh, and by the way, with this whole upstream-downstream thing, if you're headed in the downstream direction, and the first base in your transcript is called the plus-one base, there is no zero base. There's... the first base in your transcript is called plus-one. The second base in your transcript will be called plus-two, and so on. The numbers get pos... get more positive as you go downstream. To identify that we may have been talking about bases upstream of the beginning of transcription, we use negative numbers. So, around ten-ish bases away from the beginning of transcription, we find one of those hotspots. Bacterial geneticists call that the minus-ten site, because it's approximately ten bases away from the beginning of transcription. The other hotspot is even further upstream. It's called the minus 35 site, not because it's 35 bases away from the minus 10 site, but because it's about-ish 35 bases away from the plus one base. So there's a variable number of bases in between, 18, 20, stuff like that. These aren't absolute numbers, but there are two super important regions in every bacterial promoter. One of them is about 10 bases away from the start of transcription, called the minus 10 site. The other one's about 35 bases away from the start of transcription, called the minus 35 site. These sequences are extraordinarily well conserved. I used the word conserved the last time we talked when I was talking about histone proteins having conserved sequences. If you look at a histone sequence, amino acid sequence, and a whole bunch of different species, you're like, wow, that's surprisingly similar. Well, that's not a coincidence. It illustrates the importance. The fact that if you change it, it seems to have pretty negative consequences. Well, if you stack up a bunch of E. coli genes together, or even compare E. coli genes with its cousin Salmonella typhimuria, you'll see that, wow, at that minus 10 site, I seem to find mostly the same thing everywhere I look. So we say that minus 10 sites are well conserved. Minus 35 sites are well conserved. And in any one of those sequences, there is a particular set of bases that is the most likely to find at that position. The most, the best fit, if you will. So I'm going to try to turn this into a metaphor here. How many of you, by a show of hands, have a t-shirt or a sweatshirt that has a university name on it? I figured most hands would go up. In fact, I see some of them in the room right now. All right, so I would say that a college shirt is a conserved part of a college student's wardrobe. Because I probably would be able to go into most of your closets and find one, right? So a college shirt is a conserved item of clothing. But what are the letters on that shirt? Now, I see a lot of them in the room right now that say Illinois, or Fighting Illini, or something like that. But does anybody, and it's okay to be honest, anybody have a t-shirt or a sweatshirt from another school? Yeah, there's a few out there. It's not as common in this environment, in this comparison. It's not as common as the ones that say Illinois. Maybe you've got one, as long as it doesn't say Michigan, I'm okay with it. Anything but Michigan, then I will happily use that as an example. My point here is that there's a most likely, like if I was a Benton man, I'd say that if you've got a college sweatshirt in your dresser, it probably says Illinois. I'm not going to be right on that 100% of the time. So just because we say that what I'm most likely to find at the minus 10 site is a T-A-T-A-A-T, that's like saying, that's the I-L-L-I-N-O-I-S, right? That's what I'm expecting to see. It doesn't mean it doesn't say I-N-D-I-A-N-A. It could say something else. But here's the catch. If I change, did I put that right on there? Hang on a second. I haven't gotten to that. I was about to jump to a point that I haven't reached yet. So this is that lineup. This is an example of that kind of lineup. This is a dozen very different E. col And notice that they're not all perfectly lined up. That's why I said ish. 35-ish base pairs away from the beginning of transcription. 10 or so base pairs away from the beginning of transcription. But when you do find a way to line them up, look at how similar they are. This is the minus 10 site right here. A dozen different E. coli genes. What do I find at the first position of that spot, of that site? T-C-T-T-T-T-T-T-T-T. Is there a base of the four that seems overly represented at that position? Yeah, thymine. So thymine would be called the consensus base in that conserved position. Does it mean you always find a thymine there? No. There's an example in that lineup that isn't a thymine. That doesn't mean we're wrong with our consensus sequence. The consensus sequence is what you're most likely to find at any given position. What about the second position? A-A-A-A-A-G-T-A-A-A-A-A. It's not universal, but more often than not, you're going to find an A there. So an A is the consensus base at that second position of the minus 10 site. And you do that for all of the other bases in that six base really important footprint, if you will. And you come to the conclusion that more often than not, I'm going to find a T-A-T-A-A-T. That's my consensus sequence for the conserved region that we call the minus 10 site. When you do the same thing at the minus 35 site, you find that it's most likely a T-T-G-A-C-A. Those are the most likely bases to find at the minus 35 site. It doesn't mean that it's always specifically that sequence. But here's the catch. And this is what I almost jumped to a second ago. If T-T-G-A-C-A is what you're looking for, then you might recognize T-A-T-G-C-A. You're only missing one of the bases. And so it's sort of like a foot and a shoe analogy. If the foot is a defined size and the foot tries to fit into a shoe that's just a little bit too big, it's going to fit. And the foot is probably going to say, close enough, this is probably one of my shoes. Who cares if it feels a little bit loose? But the more you change the size of that shoe, the more likely it is the foot's going to be put in it and say, that's not my shoe. And it's not even going to try to be in that shoe. The more you deviate from that consensus sequence, the less effective the RNA polymerase is at recognizing you as a promoter in the first place. And if I don't recognize you as a promoter, I'm not going to carry out transcription on you. So at some point, you can change too much. But there is a benefit, believe it or not, to changing a little bit. It's not a perfect fit. Every once in a while, an RNA polymerase says, no, I don't think that's my shoe, and it moves on. That means you make less copies of that gene. Well, maybe in my 500,000 protein need example from a moment ago, let's call that protein A. What if protein B, we only need 1,000 copies of it? Well, I would want to have a stronger promoter. We call the strength of the promoter how closely it matches the consensus sequence. You'd want to have a stronger promoter in front of a gene that you need to be doing more transcription from because you need more copies of that protein. If you've got a gene over here that, yes, it's important, but I just don't need nearly as much of protein B as I do of protein A, then maybe you intentionally, and by intentionally, I mean allowing mutations to accumulate over time, maybe you allow it to become a weaker promoter, and it's not transcribed as frequently, so you don't get as much protein made from it. So it's not always a bad thing to deviate from the consensus sequence, but the more you deviate, the less transcription is going to occur from that gene. So that's sort of the important take-home point there. Another way to look for that consensus sequence is a really nice invention somewhat recently called the sequence logo. So yes, you can painstakingly identify each one of these and find where the most likely base is, or you can let computers give you a graphical representation where the relative height of the letter gives you the proportion of the time that it's found at that position. This makes it almost instantaneous to determine what the consensus sequence is of a particular conserved region. What would you think the consensus sequence of this little six-base pair region would be? There's a gigantic T. The only other thing that's even found a little bit of the time is a cytosine. There's an A most of the time, a T most of the time, an A most of the time, an A all of the time there, and a T all of the time there. So just the heights of the letters in this sequence logo tells us instantly that the consensus sequence of this would be T, A, T, A, A, T. So if you see a sequence logo on an exam, don't spend too much time on it. Just look for the biggest letters. That tells you what the consensus base is at any of those positions. Now, let's go back to, oh, yeah, please. That guy's bad. Yes, yes. Well, if it's... Really significant, statistically significantly more disproportionately represented. It tends to be put on the top. Sometimes you'll notice that there was an A that was this big and a C that was a little bit bigger underneath it. That's because if you look at the error bars, that's not necessarily a statistical difference. So less likely to be able to name a consensus base at that position, which is why we don't have consensus sequences for everything in the genome. A lot of them. Most of it is not so highly conserved. Now, let's go back to our subunits of E. coli's RNA polymerase holoenzyme. Remember, there was a sigma factor. And I'll remind you, although it's not immediately relevant, that the sigma factor is detachable. When it's there, it's called the holoenzyme. When it's gone, it's called the core enzyme. When it's there, and it's part of the holoenzyme, its job is to be the feet looking for the shoes. The shoes are the minus The feet are the parts that are, don't worry about the fact that one's labeled sigma four and the other one's labeled sigma two. The top part of this figure that's either in light or dark purple, that's the RNA polymerase molecule itself. A couple alpha subunits, a couple beta subunits that aren't labeled. There's an omega subunit that isn't even drawn up there on the top. And then in darker purple is the sigma subunit. Just like the promoter itself, which is this big but has two important parts, the sigma subunit is this big but has two important parts. Those are the feet that are looking for those shoes. One of them's looking for the TATAAT shoe. The other one's looking for the TTGACA shoe. If they don't find those shoes in that orientation, because let me tell you something else, this, what I'm doing with my legs, is different than this. And it's different than this. And it's different than this. You have to find exactly what you're looking for. That's sigma's job. Sigma factor's job is to scan unwound DNA looking for minus 10 sites and minus 35 sites. Now that raises an interesting question. If what's important is finding a TATAAT and my DNA is unwound, how can I tell whether I have found a TATAAT or not? What are your thoughts on that?  How can I tell the sequence information of what's inside the helix if I haven't unwound the helix? Is it completely inaccessible? Is the information in the helix, the nucleotide bases, on the inside of the helix, is it completely inaccessible? Why are you shaking your head no? They're called grooves, right? There's a major groove and a minor groove. And it's not a coincidence that most DNA binding proteins are scanning the major groove because that's how they can gain physical access to the sequence of bases. They can tell at a molecular level what the next base is without having to pull the strands apart. So yes, the sigma factor is non-specifically scanning DNA even before it's been unwound. So let's walk through the steps. Here's another representation. It uses different shapes. Who cares? This is the guy up here in the upper right-hand corner. That's supposed to represent the sigma factor. A molecule of the RNA polymerase holoenzyme is just out looking for work. It's that simple. RNA polymerases are looking for promoters. It either finds one or it doesn't. If it doesn't find one, it just keeps looking. It's scanning unwound DNA. The feet are looking for the slippers. And don't find anything, don't find anything, don't find anything. Oh, oh, that's a good fit right there. The sigma factor just found the minus 10 and minus 35 site and locked everything into place. Now you are, now you're not scanning anymore. The sigma factor sort of holds you in place and says, I don't think we should move right now. I think we found a promoter. So you're stable, and when you're stable you have formed what's called a closed promoter complex. We found a promoter, but we haven't even unwound anything yet. Now, do we need to recruit a separate helicase molecule? We clearly need to separate strands. I hope that's obvious. If we're going to get transcription to happen here, we've got to separate those strands. In the world of DNA replication, we recruited a separate molecule called DNA helicase. Do you see one drawn on this figure? No. What do you think that means? I'm telling you up front, we have the need for a helicase activity. I'm identifying that you don't see one in the figure. What does that suggest to you? Yes, please. Wonderful answer. The RNA polymerase does it itself. It's got a lot of subunits, right? Sigma's job is to find promoter regions. One of the other subunits' job is to unwind double-stranded DNA, only when it's been locked in place by the sigma factor feet finding the appropriate slippers. So yes, it engages its own internal helicase activity and keeps the DNA unwound within the confines of the transcript, of the polymerase itself. So you unwind as you go and you allow your trailing edge to rewind so that there's no exposed single-stranded DNA on the outsides of the footprint of the RNA polymerase. So as soon as you have found the promoter and you've reached the closed promoter complex, the next step is to separate the strands. We're gonna call that the open promoter complex. We haven't done anything nucleotide-wise yet, but we're going to. Now we go up to the, we're going from the bottom over to the second column back to the top, and the next thing that happens is I put in my first base. How can I let this hybridization happen? You'll notice I came in with a triphosphate nucleotide and I don't see any hydrolysis there. There's no pyrophosphate going off this way. This is just a hybridization. Why is this okay? Because RNA polymerase is an RNA synthesizing enzyme, and just like primase, RNA synthesizing enzymes can start a strand from scratch just by saying, what's that? Oh, there's a C in that template. Let me just hybridize a G there. Let me just let it rest there. And then I'm going to move over and I'm going to read the next space in the template. Oh, that's an adenine. Let me put in a uracil. And I'm going to covalently link it to that guanine I put in there first. Every subsequent reaction is a synthesis reaction, but the first one was just a hybridization. See, there's enough similarities between transcription and DNA replication to make it nice and confusing when you're trying to keep them straight on an exam. There's enough differences that they are not the same process, and the differences are important. But there are enough overlapping similarities that it's easy to get confused. So take the time to make sure that you're compartmentalizing the processes and not allowing them to overlap too much. Then I'm going to add in as many more bases as I can trying to move downstream. But the problem is moving downstream, if you're an RNA polymerase holoenzyme, moving downstream with your sigma factor still attached is like trying to drive with a parking brake on. Yeah, you can do it, but you're going to encounter some resistance. Why is there resistance? Because think about sigma factor's job. Sigma factor's job is to plant and say, don't move. We found one. Now you're trying to move, and sigma factor doesn't. It's not smart. It doesn't know what's going on. It's just like, no, no, no. I'm trying to tell you this is an important place. The rest of the polymerase is like, yeah, we know. We're ready to move on. So bye. It just lets go of the sigma factor. Remember, that part can be released. And that's like the rock coming out of a slingshot. As soon as you let go of the sigma factor, your parking brake has been released, and you just zip on down the line until you're told to stop. And that's called termination. So there's your process right there. Elongation occurs after initiation did. Now let's take a closer look at what's going on inside that transcription bubble, inside the footprint of the RNA polymerase itself. The purple blob looks like a jelly bean. That represents your RNA polymerase. This black arrow right here is telling us which direction the polymerase is going to be moving. It's moving from your left to your right. If I know that that's the direction the polymerase is moving, then I know which way is upstream and downstream, don't I? Downstream is in the direction of polymerase movement. Polymerases don't move upstream. And so that means going to the right is downstream. If I just aim myself back in the other direction, that would be headed upstream. The polymerase doesn't do that, but we can identify that directionality. We're incorporating nucleotides into a growing nucleic acid chain. In this figure, the dark blue and light blue strands represent the double-stranded DNA. The red strand represents the RNA transcript that we've already started. We started it a while ago. Don't know where the five-prime other—oh, I just gave away part of the answer there. We don't know where the real end of this is, probably over on the left-hand side of the screen somewhere. But what can we say about these—oh, I forgot to mention, or just a reminder, you unwind as you go, you rewind as you go, because your transcript does not stay associated with the template, unlike DNA replication. In the world of DNA replication, when you used a template molecule and allowed for leading and lagging strand synthesis, you never took that new product, the daughter strand, away from the parental strand. They stayed as a helix. In transcription, as soon as you move down the line, the transcript gets peeled away from the template so that you can rewind the DNA and it's not left unprotected. These nucleotides are going to be most accurately labeled NTPs. Notice the absence of the lowercase d. We're doing RNA synthesis here, so they would be RNA nucleotides. A's, C's, G's, and U's. Yes, please. Yes, the question is, is the RNA polymerase equally responsible for rewinding as it is for unwinding? Yes, absolutely. Not that it takes much. All you've got to do is let the strands come back together. 真正的困難是把他們分開。 他們在家中的房間中, 他們能在一起工作時, 他們會回到一起。 讓我們來看一些方向性的東西。 如果我加入了一種 nucleic acid 物質, 我總是會加入三個中間的。 這是指, 紅色的 strand 會有三個中間的在你右邊。 五個中間在你左邊。 這是產品, 這是傳統。 傳統不相同於 template strand. 從畫面的方向, 藍色 strand 是用來使用 template. 藍色 strand 是不使用 template. 藍色 strand 是我的 template 物質, 而在你左邊, 這是傳統的五個中間。 Then what would we label the light blue strand over on your left? 3 prime. So we've got a 3 prime end of the light blue strand, and we trace it all the way down to the end, and all the way down to, there's the 5 prime end of the light blue strand. And that light blue strand is the template. So we call that the template strand. The one that we're not, for this gene, using as the template strand, we're going to call the coding strand. And it is sometimes, sometimes we take those terms and interchange them and change them into logical parallels. So sometimes the coding strand is called the non-template strand, and sometimes the template strand is called the non-coding strand. You can figure out what the intent is. Most of the time when I refer to it, I refer to the template strand and the coding strand. Did that answer your question? I'm just waiting to get a hold of my time. I'm going to ask you to wait on that question until a few slides from now because that's a great question and it deserves a lengthy explanation so the question which we'll get to in a second is how does RNA polymerase know which strand it's supposed to be using right that's an excellent question because it only uses one for a given gene it only uses the dark blue strand or the light blue strand but don't forget that it all ultimately comes down to directionality I have to be reading my template strand 3 prime to 5 prime so that I can be synthesizing my new strand RNA or DNA whatever doesn't matter in the 5 prime to 3 prime direction So don't forget that in the end the coding strand, that would be the dark blue one here, is similar to the transcript. And by similar I mean it's got A's in the same places and C's in the same places and so forth. It's not identical because one has ribosecets sugar, the other one's got deoxyribose, one has thymines, the other one has uracils, but you can tell that one of them is like the other. And that coding strand and the transcript being the similar molecules, they're both anti-parallel and complementary to what we were using as the template strand. So let's play with that for a second. This is a matching question but there's only two things to match. The sequence below shows a region of double-stranded DNA located in a gene that's being transcribed. If the promoter of this gene was located over to the right somewhere, anywhere over there to the right, which strand would be the template and which strand would be the coding strand. Believe it or not, you have all of the information that you need in this question to answer this question. As long as I tell you whether the promoter region is over there or over there, you can answer this question, and I'm telling you, it's over there. Thank you. One more minute. 30 seconds. 15 seconds. Five, four, three, two, one. We'll stop. I'm going to show you how you answered without telling you what the right answer was. So how would we approach a question like this? You have to know a few things. You have to know that any time, any time, you're doing template-directed nucleic acid synthesis, and transcription falls into that category, you read your template three to five so that you can be making your product five to three. Those are the rules, and they cannot not be obeyed by the molecules. So if I tell you that this sequence is in the gene and the promoter is over there, that means our overall direction of movement has to be from right to left. So if I'm moving from right to left and I have to be reading my template three to five, the only strand I could be using as my template would be the top strand. The top strand has to be the template if my polymerase is moving from right to left, which the question informs you that it is. If the promoter's over there and this is part of the gene, I'm moving that way. So upstream, downstream stuff. So that means the top strand is the template, which means the bottom strand is the coding strand. That was about a coin toss on that one. I hope that after that explanation, it makes a little more sense. Let's do an extension of this. It's a different question. Now that you've identified that the top strand is serving as the template for this gene, what's the sequence of the transcript that you would be getting if you transcribed this region? Make sure that you've got your directionality. Make sure, just remember, anti-parallel and complementary. And the question is solvable. One more minute. 30 seconds. 15 seconds. Five. Four, three, two, one. Much better on this one. Nice recovery. So if the top strand is the template, we're looking for something that has the same sequence as the bottom strand, but with uracils instead of thymines, and make sure that you line it up right. So if the left-hand side of the bottom strand is the three-prime end, then we just have to switch this around. It would be five-prime G, U, U, U, A, G, U, U, and that's option E. So that's the kind of transcription interpretation that you're going to need to be able to do for the exams. Now, let's go to the question that was asked a few minutes ago. How does Plumarace know which of the strands it's supposed to use? It's all about the shoes and the feet, right? It's about the orientation and localization of the promoter. You can be scanning DNA non-specifically in either direction, but if you encounter a TTGACA here and a TATAAT here, and this is supposed to be the beginning, closer to your transcription start, that's not really a promoter, because you can't do this. You can't mix the feet. So there's only one way, that's why these hotspots are found in pairs. In all of these promoter regions we're going to see over the next few days, there are always two hotspots, and that's because that gives you extra insurance that you didn't accidentally interpret something as a promoter that wasn't supposed to be. That the likelihood of finding the right foot and the right slipper in that orientation with everything being right tells you exactly which direction you need to go. So we've got two promoters here. One of them is supposed to direct the polymerase over to the left, the other one is supposed to direct the polymerase over to the right. How do we know? Their minus 10 sites and minus 35 sites would be on opposite sides of this. One of them would line you up so that you would be facing, the whole RNA polymerase would be facing that way. And if I'm facing that way, I need to be reading, I'm talking about the right-facing arrow here. If that's the direction that my promoter region has oriented me, because the feet found the slippers, then I need to be reading the blue strand as my template because I have to read to the right from three prime to five prime. That means I have to read the blue strand as a template and when I do that, I make essentially a copy of the red strand with uracils instead of thymines, right? So this would be my template strand, this would be my coding strand, and this lighter red would be my transcript. If my promoter region, if instead of the feet being like this, if the feet are like this, then that orients the whole polymerase to be facing in that direction. You still have to read one of those strands three to five so that you can then make your product five to three. That's why you would...
