 Thank you. Earlier today I was running late for a meeting and somebody stopped me anyway and said, hey, what's the difference between ignorance and apathy? And I said, look, I don't know and I don't care. Thank you. Good afternoon. Happy Wednesday to you all. Welcome to MCB 150. I have some announcements to make, most of them reminders about the exam that's coming up. So the first announcement is, the reminder is, don't forget that there is an exam coming up. It's a week from tomorrow, Thursday, the 7th of March, 7 to 9 p.m. Monday at 5 p.m. This is going to be true of all of our exams. The Monday of exam week is going to be the deadline for letting us know that you need a conflict exam for some reason. And then we'll work on finding a room and a time and a proctor and we'll let you know. The exam locations are already posted in Canvas. They are the same as they were for exam one. So if you know where you took your last exam, you know where you're taking this one. Make sure that you show up on time. I will remind you that in terms of what's going to be covered on the exam, the day before the exam, we will have class, but that Wednesday material will begin exam three, or unit three material. The last bit of material that will be on your exam two, a week from tomorrow night, is going to be Monday's class. So that means we wrap up exam two content with today, Friday, and Monday. So make sure that you know what's going to be covered. Because sometimes things get broken up a little bit, it seems a little discontinuous, which reminds me, or leads to my next reminder point. Don't forget that in those sets of practice problems that I posted for exam one, several of those practice exams had content on it that at the time we hadn't seen yet, and I said, don't worry about it yet. It was the metabolism stuff. The breakpoints and where people were assessed were different when those exams were given than these. So that material, don't forget that if you're looking for practice questions on glycolysis, Krebs cycle, electron transport, reduced cofactors, that kind of stuff, that's already there. You're going to have to go back to the exam one practice sets and pick out the questions. I will have the rest of the exam two practice sets available either later tonight or sometime during the day tomorrow, but I promise no later than the end of day tomorrow for the rest of those practice problems. But if you're really itching to get in there and solve some practice problems, go back to what's labeled practice exam one, because a lot of those have metabolism questions in it, and that wasn't on your exam one, it will be on your exam two. Those are the announcements. If you've ever got questions about how the order of events goes or what the expectations are, please make sure you're talking to us. We want you to be informed with what's going on in the class. When we left off on Monday, we were talking about what intron removal entails, but I ran out of time before I got to show you some pictures. So let's see some pictures. How are things happening when we remove introns and bring the exons together? We mentioned that introns can vary a lot in length, but they have some important sequences at the boundary between an intron and an exon. We call those intron-exon junctions. And we're going to orient them, we're going to give them directionality, the introns, those junctions, but we're going to do so from the point of view of the intron. So when I talk about the 5' splice site, I mean the junction between an exon and an intron at what will be the 5' end of the intron. And when we talk about the three prime splice site, again, the frame of reference is the three prime end of the intron. So keep that kind of terminology in mind as I walk us through this figure. Remember that the machinery that's going to be cutting out the introns and joining the exons together is a system of small nuclear ribonucleoprotein complexes, SNRPs. There are a number of different SNRPs. They have different numbers of RNA molecules and different numbers of proteins based on the species. We're not going to memorize any numbers here. I'm just, I want you to understand that SNRPs are the molecules that remove introns and join exons together, and they are ribonucleoprotein complexes. Anything in biology that is made up of both an RNA component and a protein component is labeled an RNP, a ribonucleoprotein complex. This is not the first time we've seen RNPs this semester. Telomerase is an example. It doesn't have anything to do with intron removal. Just being classified as a ribonucleoprotein complex doesn't even hint at what you do. It just identifies that you yourself are made up of both an RNA component and a protein component. That's all it means to be classified as an RNP. SNRPs are also RNPs, small nuclear ribonucleoprotein. It's built into the name. And the number of SNRPs varies, but what's going to happen first is that we have to recognize the 5' splice site. And we'll do so because of some important sequences in the intron. The length and the overall sequence doesn't usually matter much. But the sequence at the splice sites, that matters a lot. It also matters that there happens to be an adenine not at the 3' splice site, but near the 3' splice site. It's called an invariant adenine, meaning it does not change. There's always going to be an adenine there, and you'll see why in just a second. So the first SNRP binds at the 5' splice site. The second SNRP binds at that invariant adenine. It binds at the A. Notice it doesn't bind the 3' splice site, not yet. Now we're going to make our first cut. Well, actually, we're going to draw the ends together, closer to each other, by assembling more SNRPs. You get those other SNRPs in there, you start bunching things, and you sort of loop out the intron, so that you bring what are going to be the ends of the exons closer together. Then we make our first cut. The first cut is going to be at the 5' splice site. We cut that junction between the 3' end of the intron over on your left and the 5' end of the exon over on your left. The 5' end of the intron we're about to cut out. You cut that boundary. Now if you just left that 5' end of the intron out to dangle, it's going to interfere with stuff. It's going to get in the way. It's not going to behave. So we have to get it out of the way. We have to make it inaccessible. So you loop it back over on itself and you covalently attach it to the adenine that you bound at that other spot where that second snort initially bound. You grab onto that adenine so that you're holding onto it, and so that when you have a free end of your intron, you can loop it back over, forming like a lasso or a lariat. This is called the lariat structure of the intron. So you loop it back around, you covalently attach it to that adenine, and now it's not going to get in the way. The only remaining steps are, well, now I'm free to cut my 3' splice site, so I do that. I release my lariat. I don't need it anymore. It's of no use to me. Since I have been pulling things in in proximity and continuing to hold onto them, as soon as my lariat is gone, I have the ends of the exons right next to each other, and I join them together covalently. And what happens to the lariat is that it's just reusable RNA nucleotides. I just chop it up using nucleases, I get those nucleotides back, and I can reuse them. I have to recharge them with some phosphate groups, but now I've got recyclable material and joined exons. That's how introns get removed and exons get joined together. What happens if you screw this up? Bad things. So if you don't recognize the right junctions, the SNRPs are still going to try to act. You just kind of get confused in what's going on. There's a family of disorders called thalassemias. They come in different varieties, but what they all have in common is that they're characterized by defective hemoglobin molecules. Thalassemias are caused by mutations in the DNA in what is going to be interpreted as an intron-exon junction. So that's how important those conserved sequences are at those junctions. If you don't have the right information in the blueprint, RNA polymerase doesn't know that that uracil really should have been an adenine. It just puts in the right base when it's doing transcription, except now it's going to lead to the intron being left at least partially in place. If you don't, if the snRNPs don't find exactly what they're looking for, then they get a little confused at the molecular level. They won't cut where they're not told to. So what ends up happening is that they find nearby sequences that look like a good fit. They're not. They weren't what was supposed to be used. You use what are called false splice sites. If I've already cut this end and I can't figure out where to cut over here, I'm gonna make a best guess. Well, in genetics, best guess is not a good plan. So you need to be doing things precisely, not hoping that you guessed right. And usually what happens is that there's either intron sequence left in your exons, which shouldn't have been, or you're missing some exon sequences because you cut too much out. However you mess up, the resulting protein, the resulting RNA that's fed to a ribosome, since a ribosome isn't going to say, hey, this isn't the same kind of protein I made last time. Why are you throwing me a curveball here? It's just a sequence of bases to the ribosome. They don't know any better, and you end up making a defective protein. So you've got to be careful with this intron removal and RNA and exon joining. In talking about a variant of this intron removal, I want to introduce what seems like a side topic. I want to ask the question, is there a relationship between the size of an organism's genome... ...and how complex it is? Well, first let's take a look at the size of some relative genomes. We've got a whole bunch of different categories of organisms in this chart, and the bar represents the range that you typically find in nature for organisms that fall into that category, and the y-axis is the genome size. Over here are the mammals, like us. We have a haploid genome of 3 billion base pairs, or what would be called 3 gigabytes worth of data on the y-axis here, so we're right down here, honestly on kind of the middle end, the middle of that purple bar right there, that's us. And mammals have a fairly narrow range of genome size, and yes, we're pretty complex. But notice this light blue bar, those are protists. The protist is a terrible classification group because it's got a huge variety of living organisms, some of which are single-celled eukaryotes like paramecia and amoeba. They are not more complex than us. They are only single-celled organisms, and yet some of them have really giant genomes. A lot of plants have really big genomes. So already you can start to see that there's probably not a direct relationship between organism complexity and genome size. Let's take a look at some more broken-down comparisons. In the prokaryotic world, is there a relationship between the genome size and the number of genes? So, yes, there is a very—look at this chart here—there's a very linear relationship here. The bigger your genome, the more genes you have, and thereby the more proteins you can make. Almost perfectly linear. The gold circles represent bacterial species. The gray circles represent archaeal species. Prokaryotes, yes. There's a direct relationship between the size of your genome and how many genes you have. It's not so clear in the eukaryotic world. I would say looking at this, it looks like a scatterplot, right? There does not seem to be a correlation between how many base pairs of information you have and how many usable genes there are. So look up here, rice. Rice have bigger genomes than humans do by a considerable amount in terms of the number of genes. We've got more base pairs. They've got more genes. Does that make them more complex? Well, I don't know about any rice that are in a classroom right now contemplating their own existence like you are. So I would say no, the rice are not more complex than us. And I would say that there's not a linear correlation between the genome size and the number of genes. So, not in the eukaryotes. There's a lot of information in this table. I'm not asking you to memorize any of it. I just want to point out a random example. Well, I'm choosing this example, but in terms of just picking some information out of this table. Look at this nematode worm right here. It has 100 million base pairs of DNA in its genome, and that genome codes for a little more than 20,000 genes. Down here is us. We have 3 billion base pairs of DNA in our haploid genome, and we have about the same number of genes. How can that be? How can you have way different genome sizes and yet approximately the same number of genes? When we first sequenced the human genome... The assumption was, and this goes back more than 20 years now, when we first sequenced the human genome, the assumption was, if we already know what proteins a human cell is capable of making, and that number was known, about 100,000. Human beings can make about 100,000-ish different polypeptide chains. So the expectation was, when we sequenced and examined the human genome, we're going to find 100,000 genes, right? But we didn't. We only found about 21,000. So it was a much lower number of genes than we initially expected to find. And there needs to be an explanation for it, because the number of genes that you find in a human is not that different than the number of genes you find in a fruit fly or a flatworm, right? And yet, there's a lot of difference in the complexity of the organisms and difference in genome sizes. Where does that complexity come from? It comes from the fact that what is fed to a ribosome is going to be about the same size. If you're making protein X, and we know that fruit flies have that same protein in them, and that nematode worms have that same protein in them, and so do we. If you compare the size of the mRNA that's fed to the ribosome, they're about the same. But the size of the genes that they come from can be wildly different. And that is because we can take our exons and put them together in different combinations. So now we're back to talking about intron removal. Why do we still have introns to deal with? I mentioned on Monday that some intron sequences provide regulatory capabilities for neighboring genes. That's one reason why introns haven't been done away with. The other is the fact that we have to get rid of them. It means we can put the exons together in different combinations. And that phenomenon, that concept, is called alternative splicing. So let's talk about alternative splicing. Just because you have all of these exons available to you doesn't mean you have to always use all of them. And if you use different combinations of exons, you make different polypeptide chains. Now, they're not going to be wildly different proteins, different versions of a protein. So my favorite analogy for this is a cookie recipe. If you want to make some cookies, there are certain ingredients, if you will. You just have to have them. You can't make cookies without sugar, eggs, flour, butter. There's things that you just have to have. But do you have to put in chocolate chips? No. Do you have to put in walnuts? No. There are optional ingredients. There are required ingredients and optional ingredients in the world of splicing. There are required exons and for some genes there are optional exons. Just because you have them at your disposal doesn't mean you have to use them. And if you use different combinations of exons, you make a different mature messenger RNA. Remember the primary transcript is just what RNA polymerase made. It's got all the introns and all the exons. Then you start removing introns, joining exons together. We're going to do a little more work to it in just a minute. And you end up with what gets fed to a ribosome. Well that thing that gets fed to a ribosome is called the mature messenger RNA. And that mature messenger RNA, if I create different mature mRNAs, I make different polypeptides. That's a pretty broad number, 50 to 90%. There's a lot of margin for error there. It depends on who you ask as to how many of the human genes are alternatively spliced. A lot of them. Let's just call it that. Not a very scientific answer or even a satisfying one. But a lot of human genes are alternatively spliced. And the number of different combinations of exons can vary. All you have to have is at least two. To be considered alternatively spliced, you need to at least make a form A and a form B of that particular protein. But some of them, there are dozens of different kinds, or hundreds, or thousands. There's a human gene that can be turned into 19,000 different polypeptides. And if you thought that was impressive, there's a drosophila, the fruit fly. There's a fruit fly gene that can be processed alternatively into 38,000 different kinds of polypeptides. 38,000 different proteins from one gene. So if we're looking for sort of averages, those are extremes. Those are not typical. Tens of thousands of different versions of a protein is not typical. What is the average in humans is three to four different versions of a protein. Now remember, we're not talking about the difference between cookies and lasagna. We're talking about the difference between a chocolate chip cookie and a peanut butter cookie. They're still cookies. It's different versions of the same thing, not wildly different things. And this is how we can make 100,000 different proteins from only 21,000 genes. Got a couple of pictures of this that basically show the same thing. Whoops, went right by that one. Try that again. Uh, that's the wrong one. Okay. There we go. So the yellow rectangles are the introns. The assumption here is we do have to remove all the introns. Alternative splicing doesn't include, oh, I'm going to leave this intron in. It's that all of the other colors in that sequence, those are the exons, we don't have to use all of them. If a gene is alternatively spliced, we can pick and choose which exons we include in the mature mRNA. But just like the cookie recipe, some of them are required. So version one of this, this one is alternatively spliced into three different versions. Version one uses exons one, two, four, five, and six. It just doesn't use three. Version two uses one, three, four, five, and six. It uses three, it just doesn't use two. Version three uses one, two, four, and six. What's the sugar, eggs, flour, butter here? And what are the chips and other decorations? Do you see exons that seem to appear in every version of the protein? Yeah, like exon 1, for example. It's in every version. That's one of the essential ingredients. That's one of the things that goes into making it that kind of a protein. But apparently, exon 2 is chocolate chips and exon 3 is peanut butter chips. I can make different versions of my cookie. I can make different versions of my protein by including this exon, but not including that other one. This next slide just shows exactly the same concept. I've got all these exons to choose from. Now they're shifted. It's a different figure, different book, different color scheme. Now the green squares, rectangles, whatever, those now represent the introns. And it's, do I need all of the seven exons? Well, this figure has already showed you which are the sugar, eggs, flour, butter, the ones that are in gold. Those are the ones that you have to have in any version of the protein that you make. But in one tissue type, this figure says, what if I was making different versions of this protein in nervous system tissue than I do in muscle tissue? Well, I need them to be slightly different because nerve cells function differently than muscle cells. So maybe I use exon 3, but not 4 in my nerve cells. I use exon 4, but not 3 in my muscle tissue. This is how we can make 100,000 different proteins, even though our genome suggests we should only be able to make 21,000 or so. So that's alternative splicing, but it's not the end of the RNA processing world. We are absolutely not done with working on these RNAs. We're talking about messenger RNAs, and we're talking about eukaryotic messenger RNAs here. So intron removal and what we're about to talk about next, it's specific to mRNAs, and it's specific to eukaryotic mRNAs. So the next thing we're going to do is we're going to dress it up. We're going to put a top hat and tails. We're going to really dress this mRNA up before we send it out into the nucleus and introduce it to a ribosome for translation. So let's talk about the top half first. Notice this 5' end of an mRNA molecule. The red strand represents the rest of the RNA. You're zooming in on the 5' end here. The 5' end, because that first base, the very first base in the transcript, is the product of a hybridization, not a synthesis reaction, we didn't need to break off its pyrophosphate group. So the 5' end of an RNA molecule has a triphosphate group on it because this first base didn't need to be covalently linked to anything. And then we've got the rest of the mRNA. So coming in to this process, we have a triphosphate group. We're gonna use a member of a different family of enzymes. If you'll recall when we talked about metabolism, I mentioned that there's a family of enzymes called kinases that add phosphate groups to things. Hopefully everybody remembers that. The family of enzymes called kinases, they phosphorylate things. There's a different family that dephosphorylates things. They're called phosphatases. And it's a weird spell word, phosphatase. It seems like it should be phosphotase, it's not. Phosphatases, they remove phosphate groups. So a member of this family conveniently called five prime, or RNA five prime triphosphatase, once again, its name tells you exactly what it's going to be doing. It's gonna be removing a phosphate from the five prime end of an RNA molecule. RNA five prime phosphatase is gonna cut off one of those phosphates. Now, the RNA molecule has two phosphates at its end. We're not done working on the five prime end here. We've just shortened the three into two. Now we're gonna bring in... A guanine, just a standard GTP. We're gonna bring in an RNA guanine and we're gonna break off two of its phosphates. Almost like it was about to be added to the end of the chain, except this is not a phosphodiester bond we're about to form. This incoming guanine, which has been drawn upside down here, this incoming GTP gets its pyrophosphate group hydrolyzed, now it's GMP, and the resulting energy released by breaking off the pyrophosphate group is used to covalently link the one remaining phosphate on this guanine to the two at the end of the RNA that we had before. So now the result is I have an upside down guanine connected to the rest of my RNA by an unusual triphosphate bridge. This is not a phosphodiester linkage. This does not violate the thou must always synthesize nucleic acid 5 to 3 rule. It doesn't violate that at all because we're not making phosphodiester bonds here. We're making what's called a triphosphate bridge. This is now going to be modified. We're going to take that guanine, where now if you'll recall from way earlier in the semester, we put numbers on the atoms in the either purine or pyrimidine rings, which the fact that they're numbered was the reason why we had to shift our terminology for the numbers of the carbons in the sugar. Instead of calling them 1, 2, 3, 4, 5, we had to call them 1 prime, 2 prime, 3 prime, because the numbers 1 through potentially 7 were already taken. On the number 7 atom in this guanine ring, we're going to add a methyl group. Okay, so now think about all the things that we've done. We've taken a guanine, we've inverted it, we've covalently linked it via a triphosphate bridge to the rest of the RNA, then we methylated it at the number 7 atom. That resulting structure is going to be called the 5 prime 7-methylguanosine cap, or 5 prime cap. That's one thing that has to happen to every mature eukaryotic messenger RNA. tRNAs don't get capped. rRNAs don't get capped. The 5 prime cap structure only gets added to eukaryotic messenger RNAs. That's what happens to one end. We're also going to work on the other end. There's going to be a poly-A tail added to the 3 prime end of this molecule. So here's RNA polymerase. Doesn't know, hasn't terminated yet, doesn't know what's happening in the part of the RNA that's already made, nor does it care. It's continuing to plow forward. Near where you're about to terminate, there's going to be a very special sequence. This sequence right here. It's AAUAAA. I don't care that you memorize that or not, but that sequence is really important because there's a protein that's going to recognize there and then about 10 to 30 nucleotides closer to where the polymerase is, it's going to cut. So you don't actually cut within that recognition sequence, but that's the landmark. The instruction here is recognize the sequence and then chop over here. Now as soon as the bulk of the RNA has been separated from the transcription bubble, that's a pretty good clue to the RNA polymerase that, well, we better wrap things up, and the gene gets terminated. Right? But now you have a fresh, three prime end of your RNA molecule, and a non-template using enzyme called poly-A polymerase. Its name tells you what it does. It adds a bunch of adenines. That's why it doesn't need to consult a template. There's no template here. I'm not consulting just a three prime end of an RNA molecule. How do I know what base to add next? I couldn't possibly know what base to add next if I'm supposed to consult a template. But if the only thing I can do is add adenines, I can do that. Because I don't need you to tell me to add an adenine next if that's the only thing I can do. So poly-A polymerase creates adenine, adenine, adenine, adenine, adenine. Up to 300 of those, at least 100, sometimes as many as 300 adenines. It's shown with an AAAA dot dot dot A to show you that's, we're not writing in all of the capital letter A's here. There could be 300 of those adenines there. That's called the poly-A tail. So what do you do to eukaryotic mRNAs? You remove the introns, join the exons together, maybe doing some alternative splicing along the way. You put a 5' cap on the 5' end, you put a poly-A tail on the 3' end, and then you're ready to export. Why? Well, you can kind of see some reasons why we might need to remove introns, but what about these other things? Why do I bother with that 5' cap structure? Because I'm about to translate this thing. And ribosomes, eukaryotic ribosomes, are built to recognize this 5' cap structure, this 7-methylguanosine cap at the 5' end. So this is why ribosomes don't accidentally try to translate tRNAs or accidentally try to translate an rRNA. They don't have 5' caps. If you don't present a 5' cap, you can't be translated because the ribosome is only looking for 5' caps. It also gives you some protection. I mentioned exonucleases before, and I mentioned the directionality of exonucleases. If you're a 5' to 3' exonuclease, you're looking to identify a nucleic acid that you could nibble away from by the presence of a free, available phosphate at the 5' end. Now, technically, this is the 5' end of this molecule, but show me an available phosphate group. There are no available phosphates. So essentially you're tricking the 5' to 3' exonucleases into thinking this is the 3' end of a molecule, and they leave you alone. This buys you some time. They're eventually going to figure it out. You're eventually going to get degraded. All RNAs eventually get degraded. But it buys you some time to actually have some protein made from this RNA that we put so much effort into making. Those are the reasons why we have 5' caps. Why do we have poly-A tails? Why do we have this poly-A tail down here? It's protection.  As well, what if you're a 3' to 5' exonuclease, and you grab onto the 3' end of a processed messenger RNA in eukaryotes? What are you nibbling away at? Adenine, adenine, adenine, adenine, adenine. You're just nibbling away at the table. Tail. You haven't gotten to anything important yet. And, I'm oversimplifying this, but that nuclease can be flicked away and recruit a fresh molecule of polyapolymerase to re-extend your tail. If you're not done with that RNA, you just get rid of the nuclease that's nibbling away from your 3' end, at the moment not getting into anything you cared about, and then you just remove it, extend your tail, buy yourself some more time. It's also important for the circularization of an mRNA molecule. It's about efficiency. We may have an opportunity to talk about that later. Just wanted to... acknowledge that there are other potential reasons why poly-A tails are important. So, the next couple of slides don't show anything new. What do we need to do? We need to remove introns, we need to join the exons together, we need to put a 5' cap, a poly-A tail. That's the eukaryotic mRNA processing outline, and that's gonna be in your notes, right? I also want to remind you where this stuff is happening. In bacteria, it happens where it can. No nucleus, no organelles, so that means transcription and ultimately translation have to happen in the cytoplasm. Well now that creates a problem, because the ribosomes are floating around in the cytoplasm. And so you don't even have an opportunity to process your RNAs before ribosomes are already finding them. So all of that 5' cap addition, the poly-A tail addition, the intron removal, that doesn't happen in bacteria, because there's no physical separation. The ribosomes are getting in there before transcription is even finished, which we will conclude Unit 2's material with a brief discussion about what's called co-transcriptional translation. In bacteria, they're making protein before transcription is even done. There's no time to do RNA processing. RNA processing only happens in eukaryotes, it only happens to the mRNAs, and it happens in the nucleus. So that what you export into the cytoplasm is a mature mRNA that a ribosome can now go look for, based on the fact that it's going to find a 5' cap and it's going to initiate translation. So that's the end of the transcription stuff. I want to put a nice little bow on that with a learning catalytics question. I'm giving you a hint here. This is not necessarily tied to anything we've talked about in the last few minutes, but it is a transcription question. The nucleotide shown below is called cortisepine triphosphate. If cortisepine triphosphate is added to an in vitro transcription reaction, this nucleotide gets added onto the growing RNA chain, but can't be extended. No more nucleotides can be added to it. The added cortisepine is always found at the 3' end of an RNA molecule, confirming that synthesis always occurs in the 5' to 3' direction. Why does this terminate transcription? Why does that particular molecule serve to terminate transcription? This is a single response, multiple choice question. Thank you. One minute, please. 30 seconds. 15 seconds. 5, 4, 3, 2, 1. We'll stop delivery and no need to build any suspense. Most of you recognize there's something weird about this nucleotide. It doesn't have a hydroxyl group with a 3 prime carbon. It has a hydroxyl group with a 2 prime carbon because it started out as a ribose. But if you don't have a 3 prime hydroxyl group to extend onto, you're the end of the chain. All of these other things either aren't true or aren't supported by the evidence. So it prevents the association of RNA polymerase and transcription factors. This is part of an incorporation process we've long since initiated. So that's not true. It irreversibly binds to the active site of RNA polymerase. There's no indication that that is the case here. There's no suggestion that that was the case. It cannot be recognized by RNA polymerase. Sure it can, because it can be incorporated. It can be added, it just can't be added to. But it does lack a 3' hydroxyl group, which is essential for being, well, for making sure that there can be a next link in the chain. Now, let's switch gears and talk about the third of our big central dogma processes. We've covered DNA replication, we've covered transcription, now let's make protein. Make an actual polypeptide chain through the process of translation. At the moment, our information is in the form of an mRNA molecule with A's, C's, G's, and U's in it. But proteins are not made of A's, C's, G's, and U's. Proteins are made of amino acids. We also have to acknowledge that we haven't said anything about how amino acids get incorporated in the first place, although I did mention how they get brought to the location. And what about the rRNAs and the tRNAs that we spent some time talking about? So I'm going to introduce this idea by asking a semi-rhetorical question. Is there a one-to-one relationship between an amino acid and a nucleotide? In other words, to put this into a more specific example, if I have a cytosine in my RNA molecule, does that mean that every time I find a cytosine, I have to put a particular amino acid at that position? Like I said glycine here, it could have been any of the 20. Well, I think we can already discount that as a possibility because if that were true, if it were true that there was a one-to-one relationship, then there could only be four amino acids specified. If each A, C, G, or U specified a singular amino acid, there could only be four amino acids. Well, we already know there's 20. So clearly, there's not a one-to-one relationship. What is that relationship? What about two? Is it a two-to-one relationship? Does C, A always specify methionine, and G, U always specify tryptophan? Is it a two-to-one relationship? Let's do some math to determine whether that's a sufficient number. If I've got four possibilities, A, C, G, U, at my first position, and four possibilities at my second position, the same A, C, G, U, that's only 16 combinations of two nucleotide sequences. That's close, but it's not close enough, because that's only 16 possibilities. We already go into this knowing that we have 20 amino acids that can be used in normal protein synthesis. What about three? The math is the same, just extending it by one. You have four possibilities of position one, four of position two, four of position three, four times four times four is 64. Now 64 is a lot farther. If we're playing the prices right here, this is, you know, 16 is closer to 20 than 64 is to 20, but you've got to get to at least 20. So 64 is the smallest number that's at least 20. Now it could theoretically be four. That's redundant. If I've already accounted for enough information to specify 20 amino acids with three, then doing four is overkill. And cells don't usually like to do that kind of overkill. So it turns out that three is the minimum number to account for the 20 amino acids that we already knew about, and then it was experimentally determined that, yeah, that's right. How was that determined? Unbelievable experiment. All right, so Marshall Nirenberg and colleagues created an in vitro protein synthesis apparatus. If you're familiar, I'm jumping ahead to something you don't have to be able to identify. But if you've heard this story before, I don't mean the Nirenberg story, I mean the protein synthesis story, you may already know what a start codon is. You may already know that methionine begins every protein. In an in vitro experiment, you can create a scenario where that doesn't have to be true. We're breaking the rules because it's not in a living cell. So Nirenberg's group said, what if I took a nucleic acid molecule that was exactly 60 uracils and I fed that molecule to an artificial protein synthesizing apparatus that doesn't have to do stop codons and start codons and stuff. You just put this in and a polypeptide comes out the other side. And what came out the other side was 20 phenylalanines. You put in exactly 60 uracils and what you get is exactly 20 phenylalanines. This proved two things. It proved that it is a triplet code. Put in 60, get out 20, that's a 3 to 1 ratio. Every three bases specifies a single amino acid. It also tells us that UUU specifies phenylalanine. And then they made different combinations of these molecules and every time they fed it in, they monitored what came out. And by the time all of these experiments were done, it sounds simpler than it is. It was an amazing experiment, or set of experiments. The take home point was, in the end, there are 61 of the 64 codons that code for amino acids. And the other three terminate polypeptide synthesis wherever they're found. And they're called stop codons. So if we only need to account for 20 amino acids, but 61 codons, a codon is a set of three bases in an RNA interpreted as a unit. If there are 61 amino acid-specifying codons, but I've only got 20 amino acids, what are the other 41 used for? It's a property of the genetic code called one of two things, either degeneracy or redundancy. Both of those words mean the same thing. It means I can have multiple codons specify a single amino acid. That's what degeneracy in the genetic code means. It means that there can be more than one codon that specifies the amino acid serine. There can be more than one codon that specifies the amino acid leucine, and so forth. It doesn't mean every codon has to be redundant or degenerate. It means that they all have that possibility. Now let me tell you what degeneracy doesn't mean. It doesn't mean that there's ambiguity in the genetic code. It is a different thing to say multiple amino acids can be specified by a single codon. That would be ambiguity than saying multiple codons can specify a single type of amino acid. That's degeneracy. One of those things is true. The other one is false. Yes, you can have multiple leucine codons. You can have multiple phenylalanine codons. But you, you, you... is always only ever going to specify phenylalanine. 如果是每次,你,你,你... If phenylalanine sometimes, 如果是每次,你,你,你... if you, you, you sometimes specify phenylalanine 如果是每次,你,你,你... and every once in a while randomly specify leucine, 那會很不容易 that would be confusing for a cell, wouldn't it? 會很不容易 That's ambiguity. There's no ambiguity. 這是不明確的,沒有明確的 The genetic code is unambiguous, but it is redundant 但它是不明確的,因為有很多 codons that specify a single amino acid. 因為有很多的 codons 指定一種酸酸 Another property of the genetic code is that it's not overlapping. 另一個特性是它不相規律 It's a string of bases that we've numbered one through whatever. 這是一個根基的根基 A codon is 123. Then the next codon is 456. 一個酸酸是123. 接著是456. Then the next codon is 789. It's not this. 接著是789. 接著是789. Codon 1 is 123. Codon 2 is 234. Codon 3 is 345. 這是不一樣的。 酸酸1是123. 酸酸2是234. Codon 3 is 345. That's overlapping. 酸酸3是345. 那是相規律 The genetic code is non-overlapping. You read three bases at a time, and then you move on to the next three bases, and then the next three bases. So it's redundant slash degenerate, but it's not ambiguous, and it's not overlapping. And we can present what three bases specify anything in the codon table by learning how to read a codon table. So let's practice. Let's take a random codon. Let's take this aspartic acid right here. What is the three letters that specify this aspartic acid? The first position is in this column over on the left. So we're going to start our codon, our three bases that specify aspartic acid, going to start with a G. The second position tells us which column we're in up here at the top. This aspartic acid is in the third column over, so that means A. The second position is a G A something. And then the third position is wherever you line up in the row on this right-hand column. So G A U intersects at that spot right there. G A U is an unambiguous codon for aspartic acid. That's how you read a codon table. I will give you a codon table. Nobody needs to memorize anything about a codon table. Do you have a question? Yes, the reading frame, which is a phrase that we'll talk about on Friday. The reading frame is preset for you, so that you're always reading three at a time. I will tell you right now something that I will mention before. The first three nucleotides in an mRNA are not going to be a start codon. You're always going to have to find the beginning and then scan in a little ways to determine where you're supposed to start. And that's where the codons read three at a time, three nucleotides at a time can be found. If you look at this codon table, you notice that most amino acids are redundant. But two of them are not. Tryptophan and methionine. For whatever reason, we'll see methionine's kind of special. But why is tryptophan only specified by one codon? I don't know, but it is. Most of them are specified by two, sometimes four, sometimes six. There are six different leucine codons. That's a lot of degeneracy or redundancy in the genetic code there. And three of them, the ones in green, are stop codons. The genetic code is, we're going, for purposes of MCB 150, is universal. What AUG specifies in E. coli, AUG specifies in humans, and everything in between. For purposes of MCB 150, the genetic code is universal. I'm saying the for purposes of MCB 150 disclaimer part because there are some unusual exceptions where sometimes this codon specifies something different. All right, we are not, not, not concerning ourselves with those exceptions. For the purpose of your exams the rest of the semester, the genetic code is universal. What UUG specifies in us, UUG specifies everywhere else along the way. All right, that's how you read a codon table. Those codons are part, you know what, that's a good place to pause. So have a great afternoon. Don't forget, I have student hours this afternoon from 4 to 5.30. Love to hear from you.
