> # Learning Goals
> 1. Describe how the synthesis of new DNA begins at an origin of replication
> 2. List the functions of Helicase, Topoisomerase, Single-strand binding protein, Primase, and DNA Polymerase at the replication Fork
> 3. Outline the Key Differences in the synthesis of the leading and lagging strands
> 4. List 3 reasons why DNA replication is very accurate
> 5. Explain how DNA replication occurs at telomeres in Eukaryotic Chromosomes
>
> We have seen the general mechanism of DNA replication (semiconservative mechanisms) and how it cooperates with the AT/GC rule
> This section explores the intricate details of DNA replication as it occurs inside living cells
> MANY Proteins are needed to initiate DNA replication so it can proceed quickly and accurately

# DNA Replication Begins at an Origin of Replication
- An **Origin of Replication** is the site within a chromosome that serves as a starting point for DNA replication.
  - At the origin, the two DNA strands Unwind, and DNA replication proceeds outwards from two replication forks in a bidirectional replication

> Image featuring the Bidirectional Replication of DNA
>
> <img width="578" alt="Screenshot 2024-02-18 at 7 40 25 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/69ea3c06-373c-406c-a89e-75f84a7f24b8">

*Note for bitch asses: Ori is Origin of Replication*

- **The number of Ori's varies among different organisms**
  - Bacteria, having small circular chromosome, have a single origin of replication
  - Bidirectional replication starts at the Ori and proceeds until the new strand meets on the opposite side of the chromosome
- Eukaryotes have larger chromosomes that are linear, and with multiple Oris, DNA can be replicated in a reasonable amount of time
  - The new strands eventually make contact with each other to complete the replication process

# DNA Replication Requires the Action of Several Different Proteins
- DNA replication happens from the Ori, called the DNA Replication Fork
- **All living species has several different proteins involved in this process**
- An understanding of the functions of these proteins is critical to explaining the replication process at the molecular level
## *Helicase, Topoisomerase, and Single-Strand Binding Proteins: Formation and Movement of the Replication Fork*
- To even act as a template for DNA replication, the strands of double helix must separate so the fork must move
- The Ori serves as a site where this separation initially occurs
- The Strand separation at each fork then moves outwards from the enzyme **DNA helicase**
  - At each fork, DNA helicase binds to one of the DNA strands and travels in the 5` to three prime direction towards the fork
  - Using ATP as energy to break the H-bonds between base pairs, they can separate the DNA strands and keep the fork moving forward
  - The actions of DNA helicase cause knots (supercoils) to form ahead. These knots are removed by **DNA Topoisomerase** (Aka DNA gyrase)
- After the two template DNA strands have separated, they must remain that way until the daughter strands have been made
- The function of **Single-strand binding proteins** is to coat both of the single strands of DNA and prevent them from re-forming a double helix
  - Keeping them exposed makes sure that we are able to keep templates for the synthesis of complementary strands

> Image of the Proteins that Form and move the replication fork
>
> <img width="359" alt="Screenshot 2024-02-18 at 8 02 28 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/1147710e-5d32-47e5-a023-b66fc92c1920">

## *DNA Polymerase and Primase: Synthesis of DNA Strands*
- **DNA Polymerase** is responsible for covalently linking nucleotides together to form DNA strands
- Arthur Kornberg found the enzyme in the 1950s, where the DNA polymerase resembles a human hand with the DNA weaving through it

> Image showcasing the Enzymatic Synthesis of DNA
>
> <img width="543" alt="Screenshot 2024-02-18 at 8 04 57 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/9d0de2b6-9f8d-4045-b976-f4c1a62173c1">

- As DNA polymerase slides along DNA, nucleotides with three phosphate groups (deoxynucleoside triphosphates), H-Bond to the exposed bases in the template strand according to the AT/GC rule
- At the catalytic site, DNA polymerase breaks bonds between the first and second phosphate then attaches the resulting nucleotide with one phosphate group (dNMP) to the 3` end of a growing strand through a phosphoester bond
- Breaking the Covalent Bond releases pyrophosphate (PPi)
  - This is an exergonic reaciton providing energy to covalently connect adjacent nucleotides
  - Then the Pyrophosphate is broken down to two phosphates
- The rate of DNA synthesis is really fast.
  - Bacteria has DNA polymerase synthesize DNA at a rate of 500 nucleotides per second, while Eukaryotic species make DNA at a rate of 50 nucleotides per second
- DNA polymerase has 2 additional enzymatic features to determine how DNA strands are made
  - 1. If DNA or RNA strand is already made to a template strand DNA polymerase can elongate this strand by making DNA
    2. If DNA polymerase is unable to begin DNA synthesis on a bare template strand, **DNA Primase** is required to make a complementary primer that is actually a short segment of RNA (10-12 nucleotides in length)
       - These short RNA strands prime the process of DNA replication
       - Another feature is DNA polymerase is that once synthesis has begun, it can only synthesize new DNA in the 5` to 3 prime direction

> Enzymatic Features of DNA Polymerase
>
> <img width="355" alt="Screenshot 2024-02-18 at 8 26 41 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/ef66d789-375d-44fb-afff-caa706450646">

## * Leading and Lagging DNA Strands Are Made Differently* 
- DNA Replication occurs near the opening that forms each replication fork
  - 1. The synthesis of a strand always begins with an RNA primer (yellow in the image below) and the new DNA is made in the 5` to 3 prime direction. How the daughter strands are made is strikingly different
         - The **leading strands** are made in the same direction as the fork movement and a one continuous molecule
         - The other **lagging strand** is a series of small fragments that are connected to form a continuous strand. The fragments are Okazaki fragments, named after Reiji and Tsueneko Okazaki discovered it in the late 1960s
         - Synthesis of Okazaki Fragments occurs in the opposite direction from the movement of the replication fork
   - Steps 2 and three are synthesized from left to right (image below)
   - Step 4 has the RNA primer eventually removed and the adjacent Okazaki fragments connected via a continuous strand of DNA

> The image below in Question (Synthesis of New DNA Strands)
>
> <img width="365" alt="Screenshot 2024-02-18 at 8 33 49 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/cf738f4c-4396-48a6-bd44-967c9adb7561">

- The image below shows how leading and lagging strands are made in bidirectional DNA replication from a single Ori.
  - The fork moving to the left uses the top template strand to make the leading strand and the bottom template strand to make the lagging strand
  - The fork moving right uses the top strand to make the lagging strand and bottom strand to make the leading strand

> Image of DNA Replication from a single origin of Replication
>
>  <img width="372" alt="Screenshot 2024-02-18 at 8 36 50 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/a2856864-531d-4892-851f-58e745e57c09">

- The synthesis of DNA in the images above shows the synthesis of DNA strands
  - The image below shows how the proteins involved in the synthesis of the leading and lagging strands of E. coli
  - This bacterium has DNA polymerase I and III responsible for DNA replication
  - The leading strand has DNA primase make RNA primer at the origin and then DNA polymerase III uses the subunit called the clamp protein to allow the enzyme to slide along the template strand without falling off (This is called processivity)

> Image showcasing the proteins involved with synthesizing the leading and lagging strands of E. Coli
> 
> <img width="334" alt="Screenshot 2024-02-18 at 8 39 00 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/4e3d8866-bcd5-4ed9-9ad2-59849566ea0e">

- The lagging strand has DNA synthesized in the 5` to 3 prime direction
- However, the synthesis occurs in a direction away from the replication fork
- The lagging strand has short segments of DNA made discontinuously as a series of Okazaki Fragments each requiring its own primer and DNA polymerase synthesizing the remainder of the fragment
- Completing the synthesizes of Okazaki fragment within the lagging strand requires 3 more events: removing RNA primers, synthesizing DNA in the area where the primers have been removed, and covalently joining adjacent fragments of DNA
  - RNA primers are removed via **DNA Polymerase I**, digesting linkages between nucleotides in each RNA primer in a 5` to 3 prime direction
  - When the RNA primer is removed DNA polymerase I fills the vacant region with DNA
  - After filling with DNA, a covalent bond is missing between the last nucleotide from DNA polymerase I and the first nucleotide in the adjacent Okazaki fragment
  - **DNA Ligase** catalyzes the formation of a covalent bond between these two DNA fragments to complete the replication process in the lagging strand

> Table Summarizing Protein Functions in DNA Replication
>
> <img width="683" alt="Screenshot 2024-02-18 at 8 46 58 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/507e5075-8928-400e-a816-5628734308b1">

# DNA Replication is VERY accurate
- Errors happen in DNA replication, but permanent mistakes are super rare
  - E.g. Bacterial DNA replication only has 1 mistake per 100 Million Nucleotides
- **high Fidelity** refers to a process occurring with few mistakes
- Why Is DNA replication High Fidelity?
  - 1. Hydrogen Bonding of A and T or G and C is more stable than mismatched pairs
    2. Active sites of DNA polymerase is unlikely to catalyze a bond between adjacent nucleotides if a mismatched base pair is formed
    3. DNA Polymerase can identify mismatched nucleotides and remove it from the daughter strand. This is **Proofreading** and occurs when DNA polymerase detects a mismatch and reverses its direction and digests the linkages between nucleotides at the end of a newly made strand in the 3` to 5 prime direction.
       - Removal of a mismatched base leads to DNA polymerase coming back to synthesize DNA in the 5` to 3 prime direction

# Evolution, Structure, and Function and how DNA Polymerases are a family of Enzymes with Specialized Functions
- 3 Important properties of DNA replication are speed, fidelity, and completeness
- DNA replication must proceed quickly and with great accuracy with no gaps between newly made strands
- To ensure these requirements, living species produce more than one DNA polymerase, each differing in rate and accuracy of DNA replication and/or the ability to prevent the formation of DNA gaps
  - Thus, our genomes have multiple genes to encode DNA polymerases made by random gene duplication events
  - Evolution has mutations that alter each gene to collectively produce a family of DNA polymerase enzymes with specialized functions
  - Natural SleectionFavors certain mutations making DNA polymerases with properties suited to the enzymes that are found
- Lets see how the DNA polymerases differ between E. coli and Humans, and how E. coli produces 5 DNA polymerase and Humans 12 (SPECIALIZATION AND FUNCTIONAL REQUIREMENTS OF EACH SPECIES)

> The he the who the huh the image
>
> <img width="641" alt="Screenshot 2024-02-18 at 9 02 29 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/1ed571b8-5400-43eb-b2fc-849d314c7fca">

- In E. coli, DNA polymerase III is responsible for most DNA replication
    - It synthesizes DNA rapidly with high fidelity
  - In Comparison, DNA polymerase I can remove RNA primers and fill in the short vacant regions with DNA
  - II, IV, and V can repair and replicate damaged DNA as I and III are stalled by DNA damage
    - These three do not stall, and with a slower rate of synthesis compared to I and III, they ensure DNA replication is complete
- In human cells DNA polymerase alpha has a "built-in" primase subunit, synthesizing RNA primers followed with short DNA regions
  - Delta and Epsilon extend DNA at a faster rate
  - Delta Polymerase synthesizes the lagging strand, and Epsilson synthesizes the leading strand
  - Gamma is in the mitochondria to replicate mitochondrial DNA
- During Replication, alpha delta or gamma may be unable to replicate over abnormal DNA structures (LESION)
- Translesion-replicating polymerases can replication over different kinds of DNA damage to ensure DNA replication is complete
- All the other DNA polymerases play an important role in DNA repair, with multiple repair enzymes arizing because of the many ways DNA gets damaged
  - MUST REPAIR DNA OR CANCER

# Telomerase Attaches DNA Sequences at the Ends of Eukaryotic Chromosomes
- At the end of eukaryotic chromosomes (**Telomeres**) has a short nucleotide sequence that is repeated in a few dozen to several hundred times
- E.g. 5-GGGTTA-3 is a sequence and has a region at the 3` end called the 3 prime overhand as it does not have a complementary strand

> Image of 3` overhand
>
> <img width="357" alt="Screenshot 2024-02-18 at 9 09 39 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/18aea7f3-e921-4382-9815-107ee47df783">

- DNA polymerase can synthesize DNA at the 5` to 3 prime direction and requires a primer
- These reasons has DNA polymerase be unable to copy the tip of a DNA strand with a 3` end.

> Image showing why this isn't possible
>
> <img width="380" alt="Screenshot 2024-02-18 at 9 10 36 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/1b0ae3af-7122-436f-9a1d-a22d664cb977">

- If this replication problem could not be overcome, linear chromosomes will be shorter with each round of DNA replication
- HOWEVER!!!! Carol Greider and Elizabeth Blackburn (1984) found telomerase that prevents chromosome shortening by attaching many copies of a DNA repeat sequence to the ends of chromosomes

> Image of Telomerase Replicating DNA 
>
> <img width="351" alt="Screenshot 2024-02-18 at 9 11 45 PM" src="https://github.com/MCBasterSheet/MCBasterSheet/assets/157755145/8b3099cd-854b-4076-8c8b-bcf8e6e9a00e">

- Telomerase has both protein and RNA
  - The RNA has a sequence complementary to the DNA repeat sequence letting it bind to the 3` overhand region of the telomere
  - After binding, the RNA beyond the binding site is a template letting telomerase synthesize a 6-nucleotide sequence at the end
  - This happens a lot, lengthening the 3` end of the DNA in the telomere, creating an upstream site for an RNA primer to be made
  - DNA polymerase then synthesizes the complementary DNA, and so the progressive shortening of chromosomes is avoided
- Telomerase function is associated with cancer
 - Cells that are cancerous divide a lot, and 90% of cancers have telomerase be at a high levels in the cancer cells
 - telomeres cannot shorten as a result, continuing growth in cancer cells
 - The mechanism whereby cancer cells are able to increase the function of telomerase is not well understood and is a topic of active research
 - Greider and Blackburb's research shared a Noble Prize in Physiology or Medicine with Jack szostak for their wok on telomeres
